<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Shea J. Andrews</title> <meta name="author" content="Shea J. Andrews"> <meta name="description" content="I have had the opportunity to contribute to 35 publications, including 12 first-author research articles, 4 first-author review articles, 17 co-authored publications, and 3 senior-authored publications"> <meta name="keywords" content="Shea Andrews, genetics, alzheimer's disease"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700%7CRoboto+Slab:100,300,400,500,700%7CMaterial+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://sjfandrews.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Shea </span>J. Andrews</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">I have had the opportunity to contribute to 35 publications, including 12 first-author research articles, 4 first-author review articles, 17 co-authored publications, and 3 senior-authored publications</p> </header> <article> <div class="publications"> <h2 class="year">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Cochran2023.png"></div> <div id="10.1002/alz.13021" class="col-sm-8"> <div class="title">Genetic associations with age at dementia onset in the PSEN1 E280A Colombian kindred</div> <div class="author"> Jesse Nicholas Cochran, Juliana Acosta‐Uribe, Bianca T. Esposito, and <span class="more-authors" title="click to view 27 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '27 more authors' ? 'Lucia Madrigal, David Aguillón, Margarita M. Giraldo, Jared W. Taylor, Joseph Bradley, Brian Fulton‐Howard, Shea J. Andrews, Natalia Acosta‐Baena, Diana Alzate, Gloria P. Garcia, Francisco Piedrahita, Hugo E. Lopez, Ashlyn G. Anderson, Ivan Rodriguez‐Nunez, Kevin Roberts, Dominantly Inherited Alzheimer Network, Devin Absher, Richard M. Myers, Gary W. Beecham, Christiane Reitz, Lindsay F. Rizzardi, Maria Victoria Fernandez, Alison M. Goate, Carlos Cruchaga, Alan E. Renton, Francisco Lopera, Kenneth S. Kosik' : '27 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">27 more authors</span> </div> <div class="periodical"> <em>Alzheimer’s &amp; Dementia</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/alz.13021" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/alz.13021" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Genetic associations with Alzheimer’s disease (AD) age at onset (AAO) could reveal genetic variants with therapeutic applications. We present a large Colombian kindred with autosomal dominant AD (ADAD) as a unique opportunity to discover AAO genetic associations. A genetic association study was conducted to examine ADAD AAO in 340 individuals with the PSEN1 E280A mutation via TOPMed array imputation. Replication was assessed in two ADAD cohorts, one sporadic early‐onset AD study and four late‐onset AD studies. 13 variants had p&lt;1×10−7 or p&lt;1×10−5 with replication including three independent loci with candidate associations with clusterin including near CLU. Other suggestive associations were identified in or near HS3ST1, HSPG2, ACE, LRP1B, TSPAN10, and TSPAN14. Variants with suggestive associations with AAO were associated with biological processes including clusterin, heparin sulfate, and amyloid processing. The detection of these effects in the presence of a strong mutation for ADAD reinforces their potentially impactful role.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2023.png"></div> <div id="10.1016/j.ebiom.2023.104511" class="col-sm-8"> <div class="title">The complex genetic architecture of Alzheimer’s disease: novel insights and future directions</div> <div class="author"> Shea J. Andrews, Alan E. Renton, Brian Fulton-Howard, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Anna Podlesny-Drabiniok, Edoardo Marcora, Alison M. Goate' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>eBioMedicine</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.ebiom.2023.104511" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ebiom.2023.104511" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background Alzheimer’s disease (AD) is a complex multifactorial neurodegenerative disorder and the most common form of dementia. AD is highly heritable, with heritability estimates of ∼70% from twin studies. Progressively larger genome-wide association studies (GWAS) have continued to expand our knowledge of AD/dementia genetic architecture. Until recently these efforts had identified 39 disease susceptibility loci in European ancestry populations. Recent developments Two new AD/dementia GWAS have dramatically expanded the sample sizes and the number of disease susceptibility loci. The first increased total sample size to 1,126,563—with an effective sample size of 332,376—by predominantly including new biobank and population-based dementia datasets. The second, expands on an earlier GWAS from the International Genomics of Alzheimer’s Project (IGAP) by increasing the number of clinically-defined AD cases/controls in addition to incorporating biobank dementia datasets, resulting in a total sample size to 788,989 and an effective sample size of 382,472. Collectively both GWAS identified 90 independent variants across 75 AD/dementia susceptibility loci, including 42 novel loci. Pathway analyses indicate the susceptibility loci are enriched for genes involved in amyloid plaque and neurofibrillary tangle formation, cholesterol metabolism, endocytosis/phagocytosis, and the innate immune system. Gene prioritization efforts for the novel loci identified 62 candidate causal genes. Many of the candidate genes from known and newly discovered loci play key roles in macrophages and highlight phagocytic clearance of cholesterol-rich brain tissue debris by microglia (efferocytosis) as a core pathogenetic hub and putative therapeutic target for AD. Where next? While GWAS in European ancestry populations have substantially enhanced our understanding of AD genetic architecture, heritability estimates from population based GWAS cohorts are markedly smaller than those from twin studies. While this missing heritability is likely due to a combination of factors, it highlights that our understanding of AD genetic architecture and genetic risk mechanisms remains incomplete. These knowledge gaps result from several underexplored areas in AD research. First, rare variants remain understudied due to methodological issues in identifying them and the cost of generating sufficiently powered whole exome/genome sequencing datasets. Second, sample sizes of non-European ancestry populations in AD GWAS remain small. Third, GWAS of AD neuroimaging and cerebrospinal fluid endophenotypes remains limited due to low compliance and high costs associated with measuring amyloid-β and tau levels and other disease-relevant biomarkers. Studies generating sequencing data, including diverse populations, and incorporating blood-based AD biomarkers are set to substantially improve our knowledge of AD genetic architecture.</p> </div> </div> </div> </li> </ol> <h2 class="year">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Harerimana2022.png"></div> <div id="10.1002/alz.12812" class="col-sm-8"> <div class="title">The role of mitochondrial genome abundance in Alzheimer’s disease</div> <div class="author"> Nadia V. Harerimana, Devashi Paliwali, Carmen Romero‐Molina, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'David A. Bennett, Judy Pa, Alison Goate, Russell H. Swerdlow, Shea J. Andrews' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Alzheimer’s &amp; Dementia</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/alz.12812" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/alz.12812" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Mitochondrial dysfunction is an early and prominent feature of Alzheimer’s disease (AD), with impaired energy metabolism preceding the onset of clinical symptoms. Here we propose an update to the mitochondrial dysfunction hypothesis of AD based on recent results examining the role of mitochondrial genome abundance in AD. In a large post mortem study, we show that lower brain mitochondrial genome abundance is associated with a greater odds of AD neuropathological change and worse cognitive performance. We hypothesize that lower mitochondrial genome abundance impairs mitochondrial function by reducing mitochondrial bioenergetics, thereby impacting neuronal and glial cell function. However, it remains to be determined if mitochondrial dysfunction causes, mediates, or is a by‐product of AD pathogenesis. Additional support for this hypothesis will be generated by linking peripheral blood mitochondrial genome abundance to AD and establishing clinical trials of compounds that upregulate total mitochondrial genome abundance or boost mitochondrial mass.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Eissman2022.png"></div> <div id="10.1093/brain/awac177" class="col-sm-8"> <div class="title">Sex differences in the genetic architecture of cognitive resilience to Alzheimer’s disease</div> <div class="author"> Jaclyn M. Eissman, Logan Dumitrescu, Emily R. Mahoney, and <span class="more-authors" title="click to view 51 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '51 more authors' ? 'Alexandra N. Smith, Shubhabrata Mukherjee, Michael L. Lee, Phoebe Scollard, Seo-Eun Choi, William S. Bush, Corinne D. Engelman, Qiongshi Lu, David W. Fardo, Emily H. Trittschuh, Jesse Mez, Catherine C. Kaczorowski, Hector Hernandez Saucedo, Keith F. Widaman, Rachel F. Buckley, Michael J. Properzi, Elizabeth C. Mormino, Hyun-Sik Yang, Theresa M. Harrison, Trey Hedden, Kwangsik Nho, Shea J. Andrews, Douglas Tommet, Niran Hadad, R. Elizabeth Sanders, Douglas M. Ruderfer, Katherine A. Gifford, Xiaoyuan Zhong, Neha S. Raghavan, Badri N. Vardarajan, Alzheimer’s Disease Neuroimaging Initiative (ADNI), Alzheimer’s Disease Genetics Consortium (ADGC), A4 Study Team, Margaret A. Pericak-Vance, Lindsay A. Farrer, Li-San Wang, Carlos Cruchaga, Gerard D. Schellenberg, Nancy J. Cox, Jonathan L. Haines, C. Dirk Keene, Andrew J. Saykin, Eric B. Larson, Reisa A. Sperling, Richard Mayeux, Michael L. Cuccaro, David A. Bennett, Julie A. Schneider, Paul K. Crane, Angela L. Jefferson, Timothy J. Hohman' : '51 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">51 more authors</span> </div> <div class="periodical"> <em>Brain</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/brain/awac177" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/brain/awac177" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Approximately 30% of elderly adults are cognitively unimpaired at time of death despite presence of Alzheimer’s disease (AD) neuropathology at autopsy. Studying individuals who are resilient to the cognitive consequences of AD neuropathology may uncover novel therapeutic targets to treat AD. It is well-established that there are sex differences in response to AD pathology, and growing evidence suggests that genetic factors may contribute to these differences. Taken together, we sought to elucidate sex-specific genetic drivers of resilience. We extended our recent large-scale genomic analysis of resilience in which we harmonized cognitive data across four cohorts of cognitive aging, in-vivo amyloid PET across two cohorts, and autopsy measures of amyloid neuritic plaque burden across two cohorts. These data were leveraged to build robust, continuous resilience phenotypes. With these phenotypes, we performed sex-stratified (N(males) = 2,093, N(females) = 2,931) and sex-interaction (N(both sexes) = 5,024) genome-wide association studies (GWAS), gene- and pathway-based tests, and genetic correlation analyses to clarify the variants, genes, and molecular pathways that relate to resilience in a sex-specific manner. Estimated among cognitively normal individuals of both sexes, resilience was 20-25% heritable, and when estimated in either sex among cognitively normal individuals, resilience was 15-44% heritable. In our GWAS, we identified a female-specific locus on chromosome 10 (rs827389, β(females) = 0.08, P(females) = 5.76E-09, β(males)=-0.01, P(males) = 0.70, β(interaction) = 0.09, P(interaction) = 1.01E-04) in which the minor allele was associated with higher resilience scores among females. This locus is located within chromatin loops that interact with promoters of genes involved in RNA processing, including GATA3. Finally, our genetic correlation analyses revealed shared genetic architecture between resilience phenotypes and other complex traits, including a female-specific association with frontotemporal dementia and male-specific associations with heart rate variability traits. We also observed opposing associations between sexes for multiple sclerosis, such that more resilient females had a lower genetic susceptibility to multiple sclerosis, and more resilient males had a higher genetic susceptibility to multiple sclerosis. Overall, we identified sex differences in the genetic architecture of resilience, identified a female-specific resilience locus, and highlighted numerous sex-specific molecular pathways that may underly resilience to AD pathology. This study illustrates the need to conduct sex-aware genomic analyses to identify novel targets that are unidentified in sex-agnostic models. Our findings support the theory that the most successful treatment for an individual with AD may be personalized based on their biological sex and genetic context.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Romero-Molina2022.png"></div> <div id="10.1016/j.neuron.2022.10.015" class="col-sm-8"> <div class="title">Microglial efferocytosis: Diving into the Alzheimer’s disease gene pool</div> <div class="author"> Carmen Romero-Molina, Francesca Garretti, Shea J. Andrews, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Edoardo Marcora, Alison M. Goate' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Neuron</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neuron.2022.10.015" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neuron.2022.10.015" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Genome-wide association studies and functional genomics studies have linked specific cell types, genes, and pathways to Alzheimer’s disease (AD) risk. In particular, AD risk alleles primarily affect the abundance or structure, and thus the activity, of genes expressed in macrophages, strongly implicating microglia (the brain-resident macrophages) in the etiology of AD. These genes converge on pathways (endocytosis/phagocytosis, cholesterol metabolism, and immune response) with critical roles in core macrophage functions such as efferocytosis. Here, we review these pathways, highlighting relevant genes identified in the latest AD genetics and genomics studies, and describe how they may contribute to AD pathogenesis. Investigating the functional impact of AD-associated variants and genes in microglia is essential for elucidating disease risk mechanisms and developing effective therapeutic approaches.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Horgusluoglu2022.png"></div> <div id="10.1002/alz.12468" class="col-sm-8"> <div class="title">Integrative metabolomics‐genomics approach reveals key metabolic pathways and regulators of Alzheimer’s disease</div> <div class="author"> Emrin Horgusluoglu, Ryan Neff, Won‐Min Song, and <span class="more-authors" title="click to view 21 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '21 more authors' ? 'Minghui Wang, Qian Wang, Matthias Arnold, Jan Krumsiek, Beatriz Galindo‐Prieto, Chen Ming, Kwangsik Nho, Gabi Kastenmüller, Xianlin Han, Rebecca Baillie, Qi Zeng, Shea Andrews, Haoxiang Cheng, Ke Hao, Alison Goate, David A. Bennett, Andrew J. Saykin, Rima Kaddurah‐Daouk, Bin Zhang, for the Alzheimer’s Disease Neuroimaging Initiative (ADNI), the Alzheimer Disease Metabolomics Consortium' : '21 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">21 more authors</span> </div> <div class="periodical"> <em>Alzheimer’s &amp; Dementia</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/alz.12468" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/alz.12468" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Metabolites, the biochemical products of the cellular process, can be used to measure alterations in biochemical pathways related to the pathogenesis of Alzheimer’s disease (AD). However, the relationships between systemic abnormalities in metabolism and the pathogenesis of AD are poorly understood. In this study, we aim to identify AD‐specific metabolomic changes and their potential upstream genetic and transcriptional regulators through an integrative systems biology framework for analyzing genetic, transcriptomic, metabolomic, and proteomic data in AD. Metabolite co‐expression network analysis of the blood metabolomic data in the Alzheimer’s Disease Neuroimaging Initiative (ADNI) shows short‐chain acylcarnitines/amino acids and medium/long‐chain acylcarnitines are most associated with AD clinical outcomes, including episodic memory scores and disease severity. Integration of the gene expression data in both the blood from the ADNI and the brain from the Accelerating Medicines Partnership Alzheimer’s Disease (AMP‐AD) program reveals ABCA1 and CPT1A are involved in the regulation of acylcarnitines and amino acids in AD. Gene co‐expression network analysis of the AMP‐AD brain RNA‐seq data suggests the CPT1A‐ and ABCA1‐centered subnetworks are associated with neuronal system and immune response, respectively. Increased ABCA1 gene expression and adiponectin protein, a regulator of ABCA1, correspond to decreased short‐chain acylcarnitines and amines in AD in the ADNI. In summary, our integrated analysis of large‐scale multiomics data in AD systematically identifies novel metabolites and their potential regulators in AD and the findings pave a way for not only developing sensitive and specific diagnostic biomarkers for AD but also identifying novel molecular mechanisms of AD pathogenesis.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Farrell2022.png"></div> <div id="10.1007/s00401-021-02379-z" class="col-sm-8"> <div class="title">Genome-wide association study and functional validation implicates JADE1 in tauopathy</div> <div class="author"> Kurt Farrell, SoongHo Kim, Natalia Han, and <span class="more-authors" title="click to view 86 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '86 more authors' ? 'Megan A. Iida, Elias M. Gonzalez, Marcos Otero-Garcia, Jamie M. Walker, Timothy E. Richardson, Alan E. Renton, Shea J. Andrews, Brian Fulton-Howard, Jack Humphrey, Ricardo A. Vialle, Kathryn R. Bowles, Katia de Paiva Lopes, Kristen Whitney, Diana K. Dangoor, Hadley Walsh, Edoardo Marcora, Marco M. Hefti, Alicia Casella, Cheick T. Sissoko, Manav Kapoor, Gloriia Novikova, Evan Udine, Garrett Wong, Weijing Tang, Tushar Bhangale, Julie Hunkapiller, Gai Ayalon, Robert R. Graham, Jonathan D. Cherry, Etty P. Cortes, Valeriy Y. Borukov, Ann C. McKee, Thor D. Stein, Jean-Paul Vonsattel, Andy F. Teich, Marla Gearing, Jonathan Glass, Juan C. Troncoso, Matthew P. Frosch, Bradley T. Hyman, Dennis W. Dickson, Melissa E. Murray, Johannes Attems, Margaret E. Flanagan, Qinwen Mao, M.-Marsel Mesulam, Sandra Weintraub, Randy L. Woltjer, Thao Pham, Julia Kofler, Julie A. Schneider, Lei Yu, Dushyant P. Purohit, Vahram Haroutunian, Patrick R. Hof, Sam Gandy, Mary Sano, Thomas G. Beach, Wayne Poon, Claudia H. Kawas, María M. Corrada, Robert A. Rissman, Jeff Metcalf, Sara Shuldberg, Bahar Salehi, Peter T. Nelson, John Q. Trojanowski, Edward B. Lee, David A. Wolk, Corey T. McMillan, C. Dirk Keene, Caitlin S. Latimer, Thomas J. Montine, Gabor G. Kovacs, Mirjam I. Lutz, Peter Fischer, Richard J. Perrin, Nigel J. Cairns, Erin E. Franklin, Herbert T. Cohen, Towfique Raj, Inma Cobos, Bess Frost, Alison Goate, Charles L. White III, John F. Crary' : '86 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">86 more authors</span> </div> <div class="periodical"> <em>Acta Neuropathologica</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1007/s00401-021-02379-z" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1007/s00401-021-02379-z" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Primary age-related tauopathy (PART) is a neurodegenerative pathology with features distinct from but also overlapping with Alzheimer disease (AD). While both exhibit Alzheimer-type temporal lobe neurofibrillary degeneration alongside amnestic cognitive impairment, PART develops independently of amyloid-β (Aβ) plaques. The pathogenesis of PART is not known, but evidence suggests an association with genes that promote tau pathology and others that protect from Aβ toxicity. Here, we performed a genetic association study in an autopsy cohort of individuals with PART (n = 647) using Braak neurofibrillary tangle stage as a quantitative trait. We found some significant associations with candidate loci associated with AD (SLC24A4, MS4A6A, HS3ST1) and progressive supranuclear palsy (MAPT and EIF2AK3). Genome-wide association analysis revealed a novel significant association with a single nucleotide polymorphism on chromosome 4 (rs56405341) in a locus containing three genes, including JADE1 which was significantly upregulated in tangle-bearing neurons by single-soma RNA-seq. Immunohistochemical studies using antisera targeting JADE1 protein revealed localization within tau aggregates in autopsy brains with four microtubule-binding domain repeats (4R) isoforms and mixed 3R/4R, but not with 3R exclusively. Co-immunoprecipitation in post-mortem human PART brain tissue revealed a specific binding of JADE1 protein to four repeat tau lacking N-terminal inserts (0N4R). Finally, knockdown of the Drosophila JADE1 homolog rhinoceros (rno) enhanced tau-induced toxicity and apoptosis in vivo in a humanized 0N4R mutant tau knock-in model, as quantified by rough eye phenotype and terminal deoxynucleotidyl transferase dUTP nick end-labeling (TUNEL) in the fly brain. Together, these findings indicate that PART has a genetic architecture that partially overlaps with AD and other tauopathies and suggests a novel role for JADE1 as a modifier of neurofibrillary degeneration.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Watts2022.png"></div> <div id="10.3233/jad-220298" class="col-sm-8"> <div class="title">A Mitochondrial DNA Haplogroup Defines Patterns of Five-Year Cognitive Change</div> <div class="author"> Amber Watts, Prabhakar Chalise, Jinxiang Hu, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Dongwei Hui, Judy Pa, Shea J Andrews, Elias K Michaelis, Russell H Swerdlow' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Journal of Alzheimer’s Disease</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.3233/jad-220298" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.3233/jad-220298" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Mitochondrial DNA (mtDNA) may play a role in Alzheimer’s disease (AD) and cognitive decline. A particular haplogroup of mtDNA, haplogroup J, has been observed more commonly in patients with AD than in cognitively normal controls. Objective: We used two mtDNA haplogroups, H and J, to predict change in cognitive performance over five years. We hypothesized that haplogroup J carriers would show less cognitive resilience. Methods: We analyzed data from 140 cognitively normal older adults who participated in the University of Kansas Alzheimer’s Disease Research Center clinical cohort between 2011 and 2020. We used factor analysis to create three composite scores (verbal memory, attention, and executive function) from 11 individual cognitive tests. We performed latent growth curve modeling to describe trajectories of cognitive performance and change adjusting for age, sex, years of education, and APOE ɛ4 allele carrier status. We compared haplogroup H, the most common group, to haplogroup J, the potential risk group. Results: Haplogroup J carriers had significantly lower baseline performance and slower rates of improvement on tests of verbal memory compared to haplogroup H carriers. We did not observe differences in executive function or attention. Conclusion: Our results reinforce the role of mtDNA in changes to cognitive function in a domain associated with risk for dementia, verbal memory, but not with other cognitive domains. Future research should investigate the distinct mechanisms by which mtDNA might affect performance on verbal memory as compared to other cognitive domains across haplogroups.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/HGI2022.png"></div> <div id="10.1038/s41586-022-04826-7" class="col-sm-8"> <div class="title">A first update on mapping the human genetic architecture of COVID-19</div> <div class="author"> COVID-19 Host Genetics Initiative,  Leadership, Gita A Pathak, and <span class="more-authors" title="click to view 901 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '901 more authors' ? 'Juha Karjalainen, Christine Stevens, Benjamin M Neale, Mark Daly, Andrea Ganna, Writing group, Writing group lead, Shea J Andrews, Masahiro Kanai, Mattia Cordioli, Analysis group, Manuscript analyses team lead, Manuscript analyses team members: PHEWAS, Renato Polimanti, Manuscript analyses team members: Mendelian randomization, Nadia Harerimana, Manuscript analyses team members: methods development, Matti Pirinen, gene prioritization, Project management group, Project management lead, Rachel G Liao, Project managment support, Karolina Chwialkowska, Amy Trankiem, Mary K Balaconis, Website development, Huy Nguyen, Matthew Solomonson, Scientific communication group, Scientific communication lead, Kumar Veerapen, Brooke Wolford, AncestryDNA COVID-19 Research Study, Analysis Team Lead, Genevieve Roberts, Data collection lead, Danny Park, Admin team lead, Catherine A Ball, Analysis team member, Marie Coignet, Shannon McCurdy, Spencer Knight, Raghavendran Partha, Brooke Rhead, Data collection member, Miao Zhang, Nathan Berkowitz, Michael Gaddis, Keith Noto, Luong Ruiz, Milos Pavlovic, Admin team member, Eurie L Hong, Kristin Rand, Ahna Girshick, Harendra Guturu, Asher Haug Baltzell, BelCovid, Analysis team lead, Mari E K Niemi, Souad Rahmouni, Julien Guntz, Yves Beguin, Sara Pigazzini, Lindokuhle Nkambule, Michel Georges, Michel Moutschen, Benoit Misset, Gilles Darcis, Julien Guiot, Samira Azarzar, Stéphanie Gofflot, Sabine Claassen, Olivier Malaise, Pascale Huynen, Christelle Meuris, Marie Thys, Jessica Jacques, Philippe Léonard, Frederic Frippiat, Jean-Baptiste Giot, Anne-Sophie Sauvage, Christian Von Frenckell, Yasmine Belhaj, Bernard Lambermont, Biobanque Quebec COVID-19, Tomoko Nakanishi, David R Morrison, Vincent Mooser, J Brent Richards, Guillaume Butler-Laporte, Vincenzo Forgetta, Rui Li, Biswarup Ghosh, Laetitia Laurent, Alexandre Belisle, Danielle Henry, Tala Abdullah, Olumide Adeleye, Noor Mamlouk, Nofar Kimchi, Zaman Afrasiabi, Nardin Rezk, Branka Vulesevic, Meriem Bouab, Charlotte Guzman, Louis Petitjean, Chris Tselios, Xiaoqing Xue, Jonathan Afilalo, Marc Afilalo, Maureen Oliveira, Bluma Brenner, Nathalie Brassard, Madeleine Durand, Erwin Schurr, Pierre Lepage, Jiannis Ragoussis, Daniel Auld, Michaël Chassé, Daniel E Kaufmann, G Mark Lathrop, Darin Adra, CCHC COVID-19 GAWS, Caroline Hayward, Joseph T Glessner, Douglas M Shaw, Archie Campbell, Marcela Morris, Hakon Hakonarson, David J Porteous, Jennifer Below, Anne Richmond, Xiao Chang, Hannah Polikowski, Petty E Lauren, Hung-Hsin Chen, Zhu Wanying, Chloe Fawns-Ritchie, Kari North, Joseph B McCormick, CHOP CAG, Joseph R Glessner, The Colorado Center for Personalized Medicine, Christopher R Gignoux, Stephen J Wicks, Kristy Crooks, Kathleen C Barnes, Michelle Daya, Jonathan Shortt, Nicholas Rafaels, Sameer Chavan, Coronagenes, Paul R H J Timmers, James F Wilson, Albert Tenesa, Shona M Kerr, Kenton D’Mellow, Egypt hgCOVID hub, Doaa Shahin, Yasser M El-Sherbiny, Kathrin Aprile von Hohenstaufen, Ali Sobh, Madonna M Eltoukhy, Lindokuhle Nkambul, Tamer A Elhadidy, Mohamed S Abd Elghafar, Jehan J El-Jawhari, Attia A S Mohamed, Marwa H Elnagdy, Amr Samir, Mahmoud Abdel-Aziz, Walid T Khafaga, Walaa M El-Lawaty, Mohamed S Torky, Mohamed R El-shanshory, Amr M Yassen, Mohamed A F Hegazy, Kamal Okasha, Mohammed A Eid, Hanteera S Moahmed, EraCORE, Carolina Medina-Gomez, M Arfan Ikram, Andre G Uitterlinden, Estonian Biobank, Reedik Mägi, Lili Milani, Andres Metspalu, Triin Laisk, Kristi Läll, Maarja Lepamets, Tõnu Esko, Ene Reimann, Paul Naaber, Edward Laane, Jaana Pesukova, Pärt Peterson, Kai Kisand, Jekaterina Tabri, Raili Allos, Kati Hensen, Joel Starkopf, Inge Ringmets, Anu Tamm, Anne Kallaste, Helene Alavere, Kristjan Metsalu, Mairo Puusepp, EXCEED, Chiara Batini, Martin D Tobin, Laura D Venn, Paul H Lee, Nick Shrine, Alexander T Williams, Anna L Guyatt, Catherine John, Richard J Packer, Altaf Ali, Robert C Free, Xueyang Wang, Louise V Wain, Edward J Hollox, Catherine E Bee, Emma L Adams, FinnGen, Aarno Palotie, Samuli Ripatti, Sanni Ruotsalainen, Kati Kristiansson, Sami Koskelainen, Markus Perola, Kati Donner, Katja Kivinen, Mari Kaunisto, Functional Host Genomics in Infectious Diseases (FHoGID), Carlo Rivolta, Pierre-Yves Bochud, Stéphanie Bibert, Noémie Boillat, Semira Gonseth Nussle, Werner Albrich, Mathieu Quinodoz, Dhryata Kamdar, Noémie Suh, Dionysios Neofytos, Véronique Erard, Cathy Voide, FHoGID, R Friolet, P Vollenweider, J L Pagani, M Oddo, F Meyer zu Bentrup, A Conen, O Clerc, O Marchetti, A Guillet, C Guyat-Jacques, S Foucras, M Rime, J Chassot, M Jaquet, R Merlet Viollet, Y Lannepoudenx, L Portopena, RegCOVID, P Y Bochud, F Desgranges, P Filippidis, B Guéry, D Haefliger, E E Kampouri, O Manuel, A Munting, M Papadimitriou-Olivgeris, J Regina, L Rochat-Stettler, V Suttels, E Tadini, J Tschopp, M Van Singer, B Viala, P-PredictUs, N Boillat-Blanco, T Brahier, O Hügli, J Y Meuwly, O Pantet, SeroCOVID, S Gonseth Nussle, M Bochud, V D’Acremont, S Estoppey Younes, CRiPSI, W C Albrich, N Suh, A Cerny, L O’Mahony, C von Mering, M Frischknecht, G-R Kleger, M Filipovic, C R Kahlert, H Wozniak, T Rochat Negro, J Pugin, K Bouras, C Knapp, T Egger, A Perret, P Montillier, C di Bartolomeo, B Barda, GCAT Genomes For Life, Rafael de Cid, Anna Carreras, Victor Moreno, Manolis Kogevinas, Iván Galván-Femenía, Natalia Blay, Xavier Farré, Lauro Sumoy, Beatriz Cortés, Josep Maria Mercader, Marta Guindo-Martinez, David Torrents, Judith Garcia-Aymerich, Gemma Castaño-Vinyals, Carlota Dobaño, GEN-COVID Multicenter Study, Marco Gori, Alessandra Renieri, Francesca Mari, Mario Umberto Mondelli, Francesco Castelli, Massimo Vaghi, Stefano Rusconi, Francesca Montagnani, Elena Bargagli, Federico Franchi, Maria Antonietta Mazzei, Luca Cantarini, Danilo Tacconi, Marco Feri, Raffaele Scala, Genni Spargi, Cesira Nencioni, Maria Bandini, Gian Piero Caldarelli, Anna Canaccini, Agostino Ognibene, Antonella D’Arminio Monforte, Massimo Girardis, Andrea Antinori, Daniela Francisci, Elisabetta Schiaroli, Pier Giorgio Scotton, Sandro Panese, Renzo Scaggiante, Matteo Della Monica, Mario Capasso, Giuseppe Fiorentino, Marco Castori, Filippo Aucella, Antonio Di Biagio, Luca Masucci, Serafina Valente, Marco Mandalà, Patrizia Zucchi, Ferdinando Giannattasio, Domenico A Coviello, Cristina Mussini, Luisa Tavecchia, Lia Crotti, Marco Rizzi, Maria Teresa La Rovere, Simona Sarzi-Braga, Maurizio Bussotti, Sabrina Ravaglia, Rosangela Artuso, Antonio Perrella, Davide Romani, Paola Bergomi, Emanuele Catena, Antonella Vincenti, Claudio Ferri, Davide Grassi, Gloria Pessina, Mario Tumbarello, Massimo Di Pietro, Ravaglia Sabrina, Sauro Luchi, Simone Furini, Simona Dei, Elisa Benetti, Nicola Picchiotti, Maurizio Sanarico, Stefano Ceri, Pietro Pinoli, Francesco Raimondi, Filippo Biscarini, Alessandra Stella, Kristina Zguro, Katia Capitani, Marco Tanfoni, Chiara Fallerini, Sergio Daga, Margherita Baldassarri, Francesca Fava, Elisa Frullanti, Floriana Valentino, Gabriella Doddato, Annarita Giliberti, Rossella Tita, Sara Amitrano, Mirella Bruttini, Susanna Croci, Ilaria Meloni, Maria Antonietta Mencarelli, Caterina Lo Rizzo, Anna Maria Pinto, Giada Beligni, Andrea Tommasi, Laura Di Sarno, Maria Palmieri, Miriam Lucia Carriero, Diana Alaverdian, Stefano Busani, Raffaele Bruno, Marco Vecchia, Mary Ann Belli, Stefania Mantovani, Serena Ludovisi, Eugenia Quiros-Roldan, Melania Degli Antoni, Isabella Zanella, Matteo Siano, Arianna Emiliozzi, Massimiliano Fabbiani, Barbara Rossetti, Laura Bergantini, Miriana D’Alessandro, Paolo Cameli, David Bennett, Federico Anedda, Simona Marcantonio, Sabino Scolletta, Susanna Guerrini, Edoardo Conticini, Bruno Frediani, Chiara Spertilli, Alice Donati, Luca Guidelli, Marta Corridi, Leonardo Croci, Paolo Piacentini, Elena Desanctis, Silvia Cappelli, Agnese Verzuri, Valentina Anemoli, Alessandro Pancrazzi, Maria Lorubbio, Federica Gaia Miraglia, Sophie Venturelli, Andrea Cossarizza, Alessandra Vergori, Arianna Gabrieli, Agostino Riva, Francesco Paciosi, Francesca Andretta, Francesca Gatti, Saverio Giuseppe Parisi, Stefano Baratti, Carmelo Piscopo, Roberta Russo, Immacolata Andolfo, Achille Iolascon, Massimo Carella, Giuseppe Merla, Gabriella Maria Squeo, Pamela Raggi, Carmen Marciano, Rita Perna, Matteo Bassetti, Maurizio Sanguinetti, Alessia Giorli, Lorenzo Salerni, Pierpaolo Parravicini, Elisabetta Menatti, Tullio Trotta, Gabriella Coiro, Fabio Lena, Enrico Martinelli, Sandro Mancarella, Chiara Gabbi, Franco Maggiolo, Diego Ripamonti, Tiziana Bachetti, Claudia Suardi, Gianfranco Parati, Giordano Bottà, Paolo Di Domenico, Ilaria Rancan, Francesco Bianchi, Riccardo Colombo, Chiara Barbieri, Donatella Acquilini, Elena Andreucci, Francesco Vladimiro Segala, Giusy Tiseo, Marco Falcone, Mirjam Lista, Monica Poscente, Oreste De Vivo, Paola Petrocelli, Alessandra Guarnaccia, Silvia Baroni, Generation Scotland, Genes &amp; Health, David A van Heel, Karen A Hunt, Richard C Trembath, Qin Qin Huang, Hilary C Martin, Dan Mason, Bhavi Trivedi, John Wright, Sarah Finer, Genes &amp; Health Research Team, Shaheen Akhtar, Mohammad Anwar, Elena Arciero, Samina Ashraf, Gerome Breen, Raymond Chung, Charles J Curtis, Maharun Chowdhury, Grainne Colligan, Panos Deloukas, Ceri Durham, Chris Griffiths, Matt Hurles, Shapna Hussain, Kamrul Islam, Ahsan Khan, Amara Khan, Cath Lavery, Sang Hyuck Lee, Robin Lerner, Daniel MacArthur, Bev MacLaughlin, Hilary Martin, Shefa Miah, Bill Newman, Nishat Safa, Farah Tahmasebi, Christopher J Griffiths, Genes for Good, Albert V Smith, Andrew P Boughton, Kevin W Li, Jonathon LeFaive, Aubrey Annis, Genome-wide assessment of the gene variants associated with severe COVID-19 phenotype in Iran, Ahmadreza Niavarani, Rasoul Aliannejad, Bahareh Sharififard, Ali Amirsavadkouhi, Zeinab Naderpour, Hengameh Ansari Tadi, Afshar Etemadi Aleagha, Saeideh Ahmadi, Seyed Behrooz Mohseni Moghaddam, Alireza Adamsara, Morteza Saeedi, Hamed Abdollahi, Abdolmajid Hosseini, pharmacogenomics, Host genetic factors in COVID-19 patients in relation to disease susceptibility, disease severity and, Pajaree Chariyavilaskul, Watsamon Jantarabenjakul, Nattiya Hirankarn, Monpat Chamnanphon, Thitima B Suttichet, Vorasuk Shotelersuk, Monnat Pongpanich, Chureerat Phokaew, Wanna Chetruengchai, Opass Putchareon, Pattama Torvorapanit, Thanyawee Puthanakit, Pintip Suchartlikitwong, Voraphoj Nilaratanakul, Pimpayao Sodsai, HUNT, Ben M Brumpton, Kristian Hveem, Cristen Willer, Wei Zhou, Tormod Rogne, Erik Solligard, Bjørn Olav Åsvold, Lifelines, Lude Franke, Marike Boezen, Patrick Deelen, Annique Claringbould, Esteban Lopera, Robert Warmerdam, Judith M Vonk, Irene van Blokland, Pauline Lanting, Anil P S Ori, Mass General Brigham-Host Vulnerability to COVID-19, Yen-Chen Anne Feng, Josep Mercader, Scott T Weiss, Elizabeth W Karlson, Jordan W Smoller, Shawn N Murphy, James B Meigs, Ann E Woolley, Robert C Green, Emma F Perez, Michigan Genomics Initiative, Sebastian Zöllner, Jiongming Wang, Andrew Beck, Mount Sinai Health System COVID-19 Genomics Initiative, Laura G Sloofman, Steven Ascolillo, Robert P Sebra, Brett L Collins, Tess Levy, Joseph D Buxbaum, Stuart C Sealfon, Daniel M Jordan, Ryan C Thompson, Kyle Gettler, Kumardeep Chaudhary, Gillian M Belbin, Michael Preuss, Clive Hoggart, Sam Choi, Slayton J Underwood, Irene Salib, Bari Britvan, Katherine Keller, Lara Tang, Michael Peruggia, Liam L Hiester, Kristi Niblo, Alexandra Aksentijevich, Alexander Labkowsky, Avromie Karp, Menachem Zlatopolsky, Marissa Zyndorf, Alexander W Charney, Noam D Beckmann, Eric E Schadt, Noura S Abul-Husn, Judy H Cho, Yuval Itan, Eimear E Kenny, Ruth J F Loos, Girish N Nadkarni, Ron Do, Paul O’Reilly, Laura M Huckins, MyCode Health Initiative, Manuel A R Ferreira, Goncalo R Abecasis, Joseph B Leader, Michael N Cantor, Anne E Justice, Dave J Carey, Geetha Chittoor, Navya Shilpa Josyula, Jack A Kosmicki, Julie E Horowitz, Aris Baras, Matthew C Gass, Ashish Yadav, Tooraj Mirshahi, Netherlands Twin Register, Jouke Jan Hottenga, Meike Bartels, Eco E J C de geus, Michel M G Nivard, Penn Medicine Biobank, Anurag Verma, Marylyn D Ritchie, Daniel Rader, Binglan Li, Shefali S Verma, Anastasia Lucas, Yuki Bradford, population, Saudi Human Genome Program - COVID19: Host Genomic markers predicting the severity and suitability to COVID-19 in highly consanguineous, Malak Abedalthagafi, Manal Alaamery, Abdulraheem Alshareef, Mona Sawaji, Salam Massadeh, Abdulaziz AlMalik, Saleh Alqahtani, Dona Baraka, Fawz Al Harthi, Ebtehal Alsolm, Leen Abu Safieh, Albandary M Alowayn, Fatimah Alqubaishi, Amal Al Mutairi, Serghei Mangul, Mansour Almutairi, Nora Aljawini, Nour Albesher, Yaseen M Arabi, Ebrahim S Mahmoud, Amin K Khattab, Roaa T Halawani, Ziab Z Alahmadey, Jehad K Albakri, Walaa A Felemban, Bandar A Suliman, Rana Hasanato, Laila Al-Awdah, Jahad Alghamdi, Deema AlZahrani, Sameera AlJohani, Hani Al-Afghani, Nouf AlDhawi, Hadeel AlBardis, Sarah Alkwai, Moneera Alswailm, Faisal Almalki, Maha Albeladi, Iman Almohammed, Eman Barhoush, Anoud Albader, Sara Alotaibi, Bader Alghamdi, Junghyun Jung, Mohammad S fawzy, May Alrashed, The genetic predisposition to severe COVID-19, Hugo Zeberg, Lindo Nkambul, Robert Frithiof, Michael Hultström, Miklos Lipcsey, Nicolas Tardif, Olav Rooyackers, Jonathan Grip, Tomislav Maricic, Study, The Norwegian Mother, Father and Child Cohort, Øyvind Helgeland, Per Magnus, Lill-Iren S Trogstad, Yunsung Lee, Jennifer R Harris, TwinsUK, Massimo Mangino, Tim D Spector, Duncan Emma, 000 Genomes Project (Genomics England), Loukas Moutsianas, Mark J Caulfield, Richard H Scott, Athanasios Kousathanas, Dorota Pasko, Susan Walker, Alex Stuckey, Christopher A Odhams, Daniel Rhodes, Tom Fowler, Augusto Rendon, Georgia Chan, Prabhu Arumugam, UK Biobank, Konrad J Karczewski, Alicia R Martin, Daniel J Wilson, Chris C A Spencer, Derrick W Crook, David H Wyllie, Anne Marie O’Connell, Elizabeth G Atkinson, Kristin Tsuo, Nikolas Baya, Patrick Turley, Rahul Gupta, Raymond K Walters, Duncan S Palmer, Gopal Sarma, Nathan Cheng, Wenhan Lu, Claire Churchhouse, Jacqueline I Goldstein, Daniel King, Cotton Seed, Mark J Daly, Hilary Finucane, Sam Bryant, F Kyle Satterstrom, Gavin Band, Sarah G Earle, Shang-Kuan Lin, Nicolas Arning, Nils Koelling, Jacob Armstrong, Justine K Rudkin, Shawneequa Callier, Caroline Cusick, UK Blood Donors Cohort, Nicole Soranzo, Jing Hua Zhao, John Danesh, Emanuele Di Angelantonio, Adam S Butterworth, Million Veteran Program, Yan V Sun, Jennifer E Huffman, Kelly Cho, Christopher J O’Donnell, Phil Tsao, J Michael Gaziano, Gina Peloso, Yuk-Lam Ho, Vanda COVID, Sandra P Smieszek, Christos Polymeropoulos, Vasilios Polymeropoulos, Mihael H Polymeropoulos, Bartlomiej P Przychodzen, study), Variability in immune response genes and severity of SARS-CoV-2 infection (INMUNGEN-CoV2, Israel Fernandez-Cadenas, Anna M Planas, Jordi Perez-Tur, Laia Llucià-Carol, Natalia Cullell, Elena Muiño, Jara Cárcel-Márquez, Marta L DeDiego, Lara Lloret Iglesias, Alex Soriano, Veronica Rico, Daiana Agüero, Josep L Bedini, Francisco Lozano, Carlos Domingo, Veronica Robles, Francisca Ruiz-Jaén, Leonardo Márquez, Juan Gomez, Eliecer Coto, Guillermo M Albaiceta, Marta García-Clemente, David Dalmau, Maria J Arranz, Beatriz Dietl, Alex Serra-Llovich, Pere Soler, Roger Colobrán, Andrea Martín-Nalda, Alba Parra Martínez, David Bernardo, Silvia Rojo, Aida Fiz-López, Elisa Arribas, Paloma de la Cal-Sabater, Tomás Segura, Esther González-Villa, Gemma Serrano-Heras, Joan Martí-Fàbregas, Elena Jiménez-Xarrié, Alicia de Felipe Mimbrera, Jaime Masjuan, Sebastian García-Madrona, Anna Domínguez-Mayoral, Joan Montaner Villalonga, Paloma Menéndez-Valladares, Women’s Genome Health Stud, Daniel I Chasman, Howard D Sesso, JoAnn E Manson, Julie E Buring, Paul M Ridker, Giulianini Franco, Phenotype steering group, Lea Davis, Sulggi Lee, James Priest, Vijay G Sankaran, David van Heel, Les Biesecker, V Eric Kerchberger, J Kenneth Baillie, COVID-19 HGI corresponding authors' : '901 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">901 more authors</span> </div> <div class="periodical"> <em>Nature</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41586-022-04826-7" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-022-04826-7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> </ol> <h2 class="year">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Lyon2021.png"></div> <div id="10.1186/s13059-020-02248-0" class="col-sm-8"> <div class="title">The variant call format provides efficient and robust storage of GWAS summary statistics</div> <div class="author"> Matthew S. Lyon, Shea J. Andrews, Ben Elsworth, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Tom R. Gaunt, Gibran Hemani, Edoardo Marcora' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Genome Biology</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13059-020-02248-0" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13059-020-02248-0" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>GWAS summary statistics are fundamental for a variety of research applications yet no common storage format has been widely adopted. Existing tabular formats ambiguously or incompletely store information about genetic variants and associations, lack essential metadata and are typically not indexed yielding poor query performance and increasing the possibility of errors in data interpretation and post-GWAS analyses. To address these issues, we adapted the variant call format to store GWAS summary statistics (GWAS-VCF) and developed open-source tools to use this format in downstream analyses. We provide open access to over 10,000 complete GWAS summary datasets converted to this format (https://gwas.mrcieu.ac.uk).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Fadista2021.png"></div> <div id="10.1016/j.ebiom.2021.103277" class="col-sm-8"> <div class="title">Shared genetic etiology between idiopathic pulmonary fibrosis and COVID-19 severity</div> <div class="author"> João Fadista, Luke M. Kraven, Juha Karjalainen, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Shea J. Andrews, Frank Geller, The COVID-19 Host Genetics Initiative, J Kenneth Baillie, Louise V. Wain, R.Gisli Jenkins, Bjarke Feenstra' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>EBioMedicine</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.ebiom.2021.103277" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ebiom.2021.103277" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background Idiopathic pulmonary fibrosis (IPF) is a complex lung disease, characterized by progressive lung scarring. Severe COVID-19 is associated with substantial pneumonitis and has a number of shared major risk factors with IPF. This study aimed to determine the genetic correlation between IPF and severe COVID-19 and assess a potential causal role of genetically increased risk of IPF on COVID-19 severity. Methods The genetic correlation between IPF and COVID-19 severity was estimated with linkage disequilibrium (LD) score regression. We performed a Mendelian randomization (MR) study for IPF causality in COVID-19. Genetic variants associated with IPF susceptibility (P&lt;5 × 10−8) in previous genome-wide association studies (GWAS) were used as instrumental variables (IVs). Effect estimates of those IVs on COVID-19 severity were gathered from the GWAS meta-analysis by the COVID-19 Host Genetics Initiative (4,336 cases &amp; 623,902 controls). Findings We detected a positive genetic correlation of IPF with COVID-19 severity (rg=0·31 [95% CI 0·04–0·57], P = 0·023). The MR estimates for severe COVID-19 did not reveal any genetic association (OR 1·05, [95% CI 0·92–1·20], P = 0·43). However, outlier analysis revealed that the IPF risk allele rs35705950 at MUC5B had a different effect compared with the other variants. When rs35705950 was excluded, MR results provided evidence that genetically increased risk of IPF has a causal effect on COVID-19 severity (OR 1·21, [95% CI 1·06–1·38], P = 4·24 × 10−3). Furthermore, the IPF risk-allele at MUC5B showed an apparent protective effect against COVID-19 hospitalization only in older adults (OR 0·86, [95% CI 0·73–1·00], P = 2·99 × 10−2) . Interpretation The strongest genetic determinant of IPF, rs35705950 at MUC5B, seems to confer protection against COVID-19, whereas the combined effect of all other IPF risk loci seem to confer risk of COVID-19 severity. The observed effect of rs35705950 could either be due to protective effects of mucin over-production on the airways or a consequence of selection bias due to (1) a patient group that is heavily enriched for the rs35705950 T undertaking strict self-isolation and/or (2) due to survival bias of the rs35705950 non-IPF risk allele carriers. Due to the diverse impact of IPF causal variants on SARS-CoV-2 infection, with a possible selection bias as an explanation, further investigation is needed to address this apparent paradox between variance at MUC5B and other IPF genetic risk factors. Funding Novo Nordisk Foundation and Oak Foundation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Paliwal2021.png"></div> <div id="10.1016/j.neurobiolaging.2021.08.005" class="col-sm-8"> <div class="title">Mitochondrial pathway polygenic risk scores are associated with Alzheimer’s Disease</div> <div class="author"> Devashi Paliwal, Tim W. McInerney, Judy Pa, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Russell H. Swerdlow, Simon Easteal, Shea J. Andrews, Alzheimer’s Disease Neuroimaging Initiative' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Neurobiology of Aging</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2021.08.005" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2021.08.005" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Genetic, animal and epidemiological studies involving biomolecular and clinical endophenotypes implicate mitochondrial dysfunction in Alzheimer’s disease (AD) pathogenesis. Polygenic risk scores (PRS) provide a novel approach to assess biological pathway-associated disease risk by combining the effects of variation at multiple, functionally related genes. We investigated the associations of PRS for genes involved in 12 mitochondrial pathways (pathway-PRS) with AD in 854 participants from Alzheimer’s Disease Neuroimaging Initiative. Pathway-PRS for the nuclear-encoded mitochondrial genome (OR: 1.99 [95% Cl: 1.70, 2.35]) and three mitochondrial pathways is significantly associated with increased AD risk: (i) response to oxidative stress (OR: 2.01 [95% Cl: 1.71, 2.38]); (ii) mitochondrial transport (OR: 1.81 [95% Cl: 1.55, 2.13]); (iii) hallmark oxidative phosphorylation (OR: 1.22 [95% Cl: 1.06, 1.40]. Therapeutic approaches targeting these pathways may have the potential for modifying AD pathogenesis. Further investigation is required to establish a causal role for these pathways in AD pathology.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/HGI2021.png"></div> <div id="10.1038/s41586-021-03767-x" class="col-sm-8"> <div class="title">Mapping the human genetic architecture of COVID-19</div> <div class="author"> COVID-19 Host Genetics Initiative, COVID-19 Host Genetics InitiativeLeadership, Mari E K Niemi, and <span class="more-authors" title="click to view 3609 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3609 more authors' ? 'Juha Karjalainen, Rachel G Liao, Benjamin M Neale, Mark Daly, Andrea Ganna, Writing group, Writing group leaders, Gita A Pathak, Shea J Andrews, Masahiro Kanai, Writing group members, Kumar Veerapen, Israel Fernandez-Cadenas, Eva C Schulte, Pasquale Striano, Minttu Marttila, Camelia Minica, Eirini Marouli, Mohd Anisul Karim, Frank R Wendt, Jeanne Savage, Laura Sloofman, Guillaume Butler-Laporte, Han-Na Kim, Stavroula Kanoni, Yukinori Okada, Jinyoung Byun, Younghun Han, Mohammed Jashim Uddin, George Davey Smith, Cristen J Willer, Joseph D Buxbaum, Analysis group, Manuscript analyses team leader, Manuscript analyses team member: meta-analysis, Juha Mehtonen, supplements, Manuscript analyses team member: heritability, methods and, Manuscript analyses team member: PHEWAS, Manuscript analyses team member: Mendelian randomization, prioritization, Manuscript analyses team member: PC projection and gene, Manuscript analyses team member: gene prioritization, Hilary Finucane, Manuscript analyses team member: sensitivity analysis, Mattia Cordioli, Manuscript analyses team members: PC projection, Alicia R Martin, Wei Zhou, In silico analysis team members, Bogdan Pasaniuc, Hanna Julienne, Hugues Aschard, Huwenbo Shi, Loic Yengo, Renato Polimanti, Maya Ghoussaini, Jeremy Schwartzentruber, Ian Dunham, Project management group, Project management leader, Project management support, Karolina Chwialkowska, Margherita Francescatto, Amy Trankiem, Mary K Balaconis, Phenotype steering group, Lea Davis, Sulggi Lee, James Priest, Alessandra Renieri, Vijay G Sankaran, David van Heel, Patrick Deelen, J Brent Richards, Tomoko Nakanishi, Les Biesecker, V Eric Kerchberger, J Kenneth Baillie, Data dictionary, Francesca Mari, Anna Bernasconi, Stefano Ceri Baillie, Arif Canakoglu, Scientific communication group, Scientific communication leaders, Brooke Wolford, Scientific communication members, Annika Faucon, Atanu Kumar Dutta, Claudia Schurmann, Emi Harry, Ewan Birney, Huy Nguyen, Jamal Nasir, Mari Kaunisto, Matthew Solomonson, Nicole Dueker, Nirmal Vadgama, Sophie Limou, Translators, Souad Rahmouni, Hamdi Mbarek, Dima Darwish, Md Mesbah Uddin, Raquel Albertos, Jordi Pérez-Tur, Ruolin Li, Lasse Folkersen, Ida Moltke, Nils Koelling, Alexander Teumer, Athanasios Kousathanas, Alicia Utrilla, Ricardo A Verdugo, Ruth Zárate, Carolina Medina-Gómez, David Gómez-Cabrero, Elena Carnero-Montoro, Carmen L Cadilla, Andrés Moreno-Estrada, Adriana Garmendia, Leire Moya, Bahar Sedaghati-Khayat, Palwendé Romuald Boua, Marie-Julie Favé, Laurent Francioli, Audrey Lemaçon, Isabelle Migeotte, Sanjay Patel, Reka Varnai, Jozsef L Szentpeteri, Csilla Sipeky, Francesca Colombo, Kathrin von Hohenstaufen, Pietro Lio, Costanza Vallerga, Qingbo Wang, Yosuke Tanigawa, Hogune Im, Chulho Han, Han Song, Jiwoo Lim, Younhe Lee, Sugyeong Kim, Sangyoon Im, Biljana Atanasovska, Hajar Fauzan Ahmad, Cindy Boer, Philip Jansen, Lude Franke, Elżbieta Kaja, Dorota Pasko, Ingrid Kennis-Szilagyi, Sergey A Kornilov, Vid Prijatelj, Ivana Prokić, Ilangkumaran Sivanadhan, Sarala Perumal, Sahar Esmaeeli, Nathaniel M Pearson, Website Development, Website development leaders, Analysis team leader, Adam Auton, Data collection leaders, Janie F Shelton, Anjali J Shastri, Analysis team members, Teresa Filshtein-Sonmez, Daniella Coker, Antony Symons, Jorge Esparza-Gordillo, Stella Aslibekyan, Jared O’Connell, Data collection members, Chelsea Ye, Catherine H Weldon, ACCOuNT, Minoli Perera, Kevin O’Leary, Matthew Tuck, Travis O’Brien, David Meltzer, Peter O’Donnell, Edith Nutescu, Guang Yang, Cristina Alarcon, Stefanie Herrmann, Sophia Mazurek, Jeff Banagan, Zacharia Hamidi, April Barbour, Noora Raffat, Diana Moreno, Admin team member, Paula Friedman, Amsterdam UMC COVID Study Group, Bart Ferwerda, Diederik van de Beek, Matthijs C Brouwer, Alexander P J Vlaar, W Joost Wiersinga, Danielle Posthuma, Elleke Tissink, A H Koos Zwinderman, Emil Uffelmann, Michiel van Agtmael, Anne Geke Algera, Frank van Baarle, Diane Bax, Martijn Beudel, Harm Jan Bogaard, Marije Bomers, Peter I Bonta, Lieuwe Bos, Michela Botta, Justin de Brabander, Godelieve de Bree, Sanne de Bruin, Marianna Bugiani, Esther Bulle, Osoul Chouchane, Alex Cloherty, Dave Dongelmans, Paul Elbers, Lucas Fleuren, Suzanne Geerlings, Bart Geerts, Theo Geijtenbeek, Armand Girbes, Bram Goorhuis, Martin P Grobusch, Florianne Hafkamp, Laura Hagens, Jorg Hamann, Vanessa Harris, Robert Hemke, Sabine M Hermans, Leo Heunks, Markus Hollmann, Janneke Horn, Joppe W Hovius, Menno D de Jong, Rutger Koning, Niels van Mourik, Jeannine Nellen, Esther J Nossent, Frederique Paulus, Edgar Peters, Tom van der Poll, Bennedikt Preckel, Jan M Prins, Jorinde Raasveld, Tom Reijnders, Michiel Schinkel, Marcus J Schultz, Alex Schuurman, Kim Sigaloff, Marry Smit, Cornelis S Stijnis, Willemke Stilma, Charlotte Teunissen, Patrick Thoral, Anissa Tsonas, Marc van der Valk, Denise Veelo, Heder de Vries, Michèle van Vugt, Dorien Wouters, René P Minnaar, Adrie Kromhout, Kees W J van Uffelen, Ruud A Wolterman, AncestryDNA COVID-19 Research Study, Genevieve Roberts, Data collection leader, Danny Park, Admin team leader, Catherine A Ball, Marie Coignet, Shannon McCurdy, Spencer Knight, Raghavendran Partha, Brooke Rhead, Miao Zhang, Nathan Berkowitz, Michael Gaddis, Keith Noto, Luong Ruiz, Milos Pavlovic, Admin team members, Eurie L Hong, Kristin Rand, Ahna Girshick, Harendra Guturu, Asher Haug Baltzell, BelCovid, Julien Guntz, Yves Beguin, Sara Pigazzini, Lindokuhle Nkambule, Youssef Bouysran, Adeline Busson, Xavier Peyrassol, Françoise Wilkin, Bruno Pichon, Guillaume Smits, Isabelle Vandernoot, Jean-Christophe Goffard, Michel Georges, Michel Moutschen, Benoit Misset, Gilles Darcis, Julien Guiot, Laurent Jadot, Samira Azarzar, Patricia Dellot, Stéphanie Gofflot, Sabine Claassen, Axelle Bertrand, Gilles Parzibut, Mathilde Clarinval, Catherine Moermans, Olivier Malaise, Kamilia El Kandoussi, Raphaël Thonon, Pascale Huynen, Alyssia Mesdagh, Sofia Melo, Nicolas Jacques, Emmanuel Di Valentin, François Giroule, Alice Collignon, Coraline Radermecker, Marielle Lebrun, Hélène Perée, Samuel Latour, Olivia Barada, Judit Sanchez, Claire Josse, Bouchra Boujemla, Margot Meunier, Emeline Mariavelle, Sandy Anania, Hélène Gazon, Danusia Juszczak, Marjorie Fadeur, Séverine Camby, Christelle Meuris, Marie Thys, Jessica Jacques, Monique Henket, Philippe Léonard, Frederic Frippiat, Jean-Baptiste Giot, Anne-Sophie Sauvage, Christian Von Frenckell, Myriam Mni, Marie Wéry, Alicia Staderoli, Yasmine Belhaj, Bernard Lambermont, Biobanque Quebec COVID-19, David R Morrison, Admin team leaders, Vincent Mooser, Vincenzo Forgetta, Rui Li, Biswarup Ghosh, Laetitia Laurent, Alexandre Belisle, Danielle Henry, Tala Abdullah, Olumide Adeleye, Noor Mamlouk, Nofar Kimchi, Zaman Afrasiabi, Nardin Rezk, Branka Vulesevic, Meriem Bouab, Charlotte Guzman, Louis Petitjean, Chris Tselios, Xiaoqing Xue, Jonathan Afilalo, Marc Afilalo, Maureen Oliveira, Bluma Brenner, Nathalie Brassard, Madeleine Durand, Erwin Schurr, Pierre Lepage, Jiannis Ragoussis, Daniel Auld, Michaël Chassé, Daniel E Kaufmann, G Mark Lathrop, Darin Adra, BioVU, Analysis team leaders, Lea K Davis, Nancy J Cox, Jennifer E Below, Julia M Sealock, Annika B Faucon, Megan M Shuey, Hannah G Polikowsky, Lauren E Petty, Douglas M Shaw, Hung-Hsin Chen, Wanying Zhu, Bonn Study of COVID-19 Genetics, Kerstin U Ludwig, Julia Schröder, Carlo Maj, Selina Rolker, Markus M Nöthen, Julia Fazaal, Verena Keitel, Björn-Erik Ole Jensen, Torsten Feldt, Ingo Kurth, Nikolaus Marx, Michael Dreher, Isabell Pink, Markus Cornberg, Thomas Illig, Clara Lehmann, Philipp Schommers, Max Augustin, Jan Rybniker, Lisa Knopp, Thomas Eggermann, Sonja Volland, Janine Altmüller, Marc M Berger, Thorsten Brenner, Anke Hinney, Oliver Witzke, Robert Bals, Christian Herr, Nicole Ludwig, Jörn Walter, CHRIS, Christian Fuchsberger, Cristian Pattaro, Alessandro De Grandi, Peter Pramstaller, David Emmert, Roberto Melotti, Luisa Foco, Deborah Mascalzoni, Martin Gögele, Francisco Domingues, Andrew Hicks, Colorado Center for Personalized Medicine (CCPM), Christopher R Gignoux, Stephen J Wicks, Kristy Crooks, Kathleen C Barnes, Michelle Daya, Jonathan Shortt, Nicholas Rafaels, Sameer Chavan, Columbia University COVID-19 Biobank, David B Goldstein, Krzysztof Kiryluk, Soumitra Sengupta, Wendy Chung, Muredach P Reilly, Atlas Khan, Chen Wang, Gundula Povysil, Nitin Bhardwaj, Ali G Gharavi, Iuliana Ionita-Laza, Ning Shang, Sheila M O’Byrne, Renu Nandakumar, Amritha Menon, Yat S So, Eldad Hod, Danielle Pendrick, Corea (Genetics of COVID-19-related Manifestation), Soo-Kyung Park, Hyung-Lae Kim, Chang Kyung Kang, Hyo-Jung Lee, Kyoung-Ho Song, Kyung Jae Yoon, Nam-Jong Paik, Woojin Seok, Heejun Yoon, Eun-Jeong Joo, Yoosoo Chang, Seungho Ryu, Wan Beom Park, Jeong Su Park, Kyoung Un Park, Sin Young Ham, Jongtak Jung, Eu Suk Kim, Hong Bin Kim, COVID-19-Hostage, David Ellinghaus, Frauke Degenhardt, Mario Cáceres, Simonas Juzenas, Tobias L Lenz, Agustín Albillos, Antonio Julià, Bettina Heidecker, Federico Garcia, Florian Kurth, Florian Tran, Frank Hanses, Heinz Zoller, Jan C Holter, Javier Fernández, Leif Erik Sander, Philip Rosenstiel, Philipp Koehler, Rafael de Cid, Rosanna Asselta, Stefan Schreiber, Ute Hehr, Daniele Prati, Guido Baselli, Luca Valenti, Luis Bujanda, Jesus M Banales, Stefano Duga, Mauro D’Amato, Manuel Romero-Gómez, Maria Buti, Pietro Invernizzi, Andre Franke, Johannes R Hov, Tom H Karlsen, Trine Folseraas, Douglas Maya-Miles, Ana Teles, Clinton Azuure, Eike Matthias Wacker, Florian Uellendahl-Werth, Hesham ElAbd, Jatin Arora, Jon Lerga-Jaso, Lars Wienbrandt, Malte Christoph Rühlemann, Mareike Wendorff, May Sissel Vadla, Ole Bernt Lenning, Onur Özer, Ronny Myhre, Soumya Raychaudhuri, Anja Tanck, Christoph Gassner, Georg Hemmrich-Stanisak, Jan Kässens, Maria E Figuera Basso, Martin Schulzky, Michael Wittig, Nicole Braun, Tanja Wesse, Wolfgang Albrecht, Xiaoli Yi, Aaron Blandino Ortiz, Adolfo Garrido Chercoles, Agustín Ruiz, Alberto Mantovani, Aleksander Rygh Holten, Alena Mayer, Alessandro Cherubini, Alessandro Protti, Alessio Aghemo, Alessio Gerussi, Alfredo Ramirez, Alice Braun, Ana Barreira, Ana Lleo, Anders Benjamin Kildal, Andreas Glück, Anna Carreras Nolla, Anna Latiano, Anne Ma Dyrhol-Riise, Antonio Muscatello, Antonio Voza, Ariadna Rando-Segura, Aurora Solier, Banasik Karina, Beatriz Cortes, Beatriz Mateos, Beatriz Nafria-Jimenez, Benedikt Schaefer, Carla Bellinghausen, Carlos Ferrando, Carmen Quereda, Carsten Skurk, Charlotte Thibeault, Christoph D Spinner, Christoph Lange, Cinzia Hu, Claudio Cappadona, Cristiana Bianco, Cristina Sancho, Dag Arne Lihaug Hoff, Daniela Galimberti, David Jiménez, David Pestaña, David Toapanta, Elena Azzolini, Elio Scarpini, Elisa T Helbig, Eloisa Urrechaga, Elvezia Maria Paraboschi, Emanuele Pontali, Enric Reverter, Enrique Navas, Eunate Arana, Félix García Sánchez, Ferruccio Ceriotti, Francesco Malvestiti, Francisco Mesonero, Gianni Pezzoli, Giuseppe Lamorte, Holger Neb, Ilaria My, Isabel Hernández, Itziar de Rojas, Iván Galván-Femenia, Jan Heyckendorf, Joan Ramon Badia, Jochen Schneider, Josune Goikoetxea, Julia Kraft, Karl Erik Müller, Karoline I Gaede, Koldo Garcia-Etxebarria, Kristian Tonby, Lars Heggelund, Laura Izquierdo-Sanchez, Lauro Sumoy, Lena J Lippert, Leonardo Terranova, Lucia Garbarino, Luis Téllez, Luisa Roade, Mahnoosh Ostadreza, Maider Intxausti, Manolis Kogevinas, María A Gutiérrez-Stampa, Maria J G T Vehreschild, Marta Marquié, Massimo Castoldi, Maurizio Cecconi, Mercè Boada, Michael J Seilmaier, Michela Mazzocco, Miguel Rodríguez-Gandía, Natale Imaz Ayo, Natalia Blay, Nilda Martínez, Norwegian SARS-CoV-2 Study Group*, Oliver A Cornely, Orazio Palmieri, Paolo Tentorio, Pedro M Rodrigues, Pedro P España, Per Hoffmann, Petra Bacher, Phillip Suwalski, Raúl de Pablo, Rosa Nieto, Salvatore Badalamenti, Sandra Ciesek, Sara Bombace, Sibylle Wilfling, Søren Brunak, Stefanie Heilmann-Heimbach, The Humanitas COVID-19 Task Force*, The Humanitas Gavazzeni COVID-19 Task Force*, Stephan Ripke, Thomas Bahmer, Ulf Landmesser, Ulrike Protzer, Valeria Rimoldi, Vegard Skogen, Victor Andrade, Victor Moreno, Wolfgang Poller, Xavier Farre, Xiaomin Wang, Yascha Khodamoradi, Zehra Karadeniz, Adolfo de Salazar, Adriana Palom, Alba-Estela Garcia-Fernandez, Albert Blanco-Grau, Alberto Zanella, Alessandra Bandera, Almut Nebel, Andrea Biondi, Andrea Caballero-Garralda, Andrea Gori, Andreas Lind, Anna Ludovica Fracanzani, Anna Peschuck, Antonio Pesenti, Carmen de la Horra, Chiara Milani, Cinzia Paccapelo, Claudio Angelini, Cristina Cea, Eduardo Muñiz-Diaz, Elena Sandoval, Enrique J Calderón, Erik Solligård, Fátima Aziz, Filippo Martinelli-Boneschi, Flora Peyvandi, Francesco Blasi, Francisco J Medrano, Francisco Rodriguez-Frias, Fredrik Müller, Giacomo Grasselli, Giorgio Costantino, Giulia Cardamone, Giuseppe Foti, Giuseppe Matullo, Hayato Kurihara, Jan Egil Afset, Jan Kristian Damås, Javier Ampuero, Javier Martín, Jeanette Erdmann, Jonas Bergan, Siegfried Goerg, Jose Ferrusquía-Acosta, Jose Hernández Quero, Juan Delgado, Juan M Guerrero, Kari Risnes, Laura Rachele Bettini, Leticia Moreira, Lise Tuset Gustad, Luigi Santoro, Luigia Scudeller, Mar Riveiro-Barciela, Marco Schaefer, Maria Carrabba, Maria G Valsecchi, María Hernandez-Tejero, Marialbert Acosta-Herrera, Mariella D’Angiò, Marina Baldini, Marina Cazzaniga, Michele Ciccarelli, Monica Bocciolone, Monica Miozzo, Natalia Chueca, Nicola Montano, Paola Faverio, Paoletta Preatoni, Paolo Bonfanti, Paolo Omodei, Pedro Castro, Ricard Ferrer, Roberta Gualtierotti, Rocío Gallego-Durán, Rubén Morilla, Sammra Haider, Sara Marsal, Serena Aneli, Serena Pelusi, Silvano Bosari, Stefano Aliberti, Susanne Dudman, Tenghao Zheng, Tomas Pumarola, Trinidad Gonzalez Cejudo, Valter Monzani, Vicente Friaza, Wolfgang Peter, Ximo Dopazo, Norwegian SARS-CoV-2 Study Group, Humanitas COVID-19 Task Force, The Humanitas Gavazzeni COVID-19 Task Force, Sandra May, Marit M Grimsrud, Daniel F Gudbjartsson, Kari Stefansson, Patrick Sulem, Gardar Sveinbjornsson, Pall Melsted, Gudmundur Norddahl, Kristjan Helgi Swerford Moore, Unnur Thorsteinsdottir, Hilma Holm, Determining the Molecular Pathways &amp; Genetic Predisposition of the Acute Inflammatory Process Caused by SARS-CoV-2, Marta E Alarcón-Riquelme, David Bernardo, Analysis team member, Manuel Martínez-Bueno, Data collection member, Silvia Rojo Rello, Estonian Biobank, Reedik Magi, Lili Milani, Andres Metspalu, Triin Laisk, Kristi Läll, Maarja Lepamets, Tõnu Esko, Ene Reimann, Paul Naaber, Edward Laane, Jaana Pesukova, Pärt Peterson, Kai Kisand, Jekaterina Tabri, Raili Allos, Kati Hensen, Joel Starkopf, Inge Ringmets, Anu Tamm, Anne Kallaste, Helene Alavere, Kristjan Metsalu, Mairo Puusepp, FinnGen, Kati Kristiansson, Sami Koskelainen, Markus Perola, Kati Donner, Katja Kivinen, Aarno Palotie, FinnGen Admin team leader, Functional Host Genomics in Infectious Diseases (FHoGID), Carlo Rivolta, Pierre-Yves Bochud, Stéphanie Bibert, Noémie Boillat, Semira Gonseth Nussle, Werner Albrich, Mathieu Quinodoz, Dhryata Kamdar, Noémie Suh, Dionysios Neofytos, Véronique Erard, Cathy Voide, FHoGID*, RegCOVID*, P-PredictUs*, SeroCOVID*, CRiPSI*, FHoGID, P Y Bochud, C Rivolta, S Bibert, M Quinodoz, D Kamdar, D Neofytos, V Erard, C Voide, R Friolet, P Vollenweider, J L Pagani, M Oddo, F Meyer zu Bentrup, A Conen, O Clerc, O Marchetti, A Guillet, C Guyat-Jacques, S Foucras, M Rime, J Chassot, M Jaquet, R Merlet Viollet, Y Lannepoudenx, L Portopena, RegCOVID, F Desgranges, P Filippidis, B Guéry, D Haefliger, E E Kampouri, O Manuel, A Munting, M Papadimitriou-Olivgeris, J Regina, L Rochat-Stettler, V Suttels, E Tadini, J Tschopp, M Van Singer, B Viala, P-PredictUs, N Boillat-Blanco, T Brahier, O Hügli, J Y Meuwly, O Pantet, SérocoViD, S Gonseth Nussle, M Bochud, V D’Acremont, S Estoppey Younes, CRiPSI, W C Albrich, N Suh, A Cerny, L O’Mahony, C von Mering, M Frischknecht, G -R Kleger, M Filipovic, C R Kahlert, H Wozniak, T Rochat Negro, J Pugin, K Bouras, C Knapp, T Egger, A Perret, P Montillier, C di Bartolomeo, B Barda, GCAT Genomes For Life, Anna Carreras, Iván Galván-Femenía, Xavier Farré, Beatriz Cortés, Josep Maria Mercader, Marta Guindo-Martinez, David Torrents, Judith Garcia-Aymerich, Gemma Castaño-Vinyals, Carlota Dobaño, GEN-COVID Multicenter Study, Marco Gori, Mario Umberto Mondelli, Francesco Castelli, Massimo Vaghi, Stefano Rusconi, Francesca Montagnani, Elena Bargagli, Federico Franchi, Maria Antonietta Mazzei, Luca Cantarini, Danilo Tacconi, Marco Feri, Raffaele Scala, Genni Spargi, Cesira Nencioni, Maria Bandini, Gian Piero Caldarelli, Maurizio Spagnesi, Anna Canaccini, Agostino Ognibene, Antonella D’Arminio Monforte, Massimo Girardis, Andrea Antinori, Daniela Francisci, Elisabetta Schiaroli, Pier Giorgio Scotton, Sandro Panese, Renzo Scaggiante, Matteo Della Monica, Mario Capasso, Giuseppe Fiorentino, Marco Castori, Filippo Aucella, Antonio Di Biagio, Luca Masucci, Serafina Valente, Marco Mandalà, Patrizia Zucchi, Ferdinando Giannattasio, Domenico A Coviello, Cristina Mussini, Giancarlo Bosio, Luisa Tavecchia, Lia Crotti, Marco Rizzi, Maria Teresa La Rovere, Simona Sarzi-Braga, Maurizio Bussotti, Sabrina Ravaglia, Rosangela Artuso, Antonio Perrella, Davide Romani, Paola Bergomi, Emanuele Catena, Antonella Vincenti, Claudio Ferri, Davide Grassi, Gloria Pessina, Mario Tumbarello, Massimo Di Pietro, Ravaglia Sabrina, Sauro Luchi, Chiara Barbieri, Donatella Acquilini, Elena Andreucci, Francesco Paciosi, Francesco Vladimiro Segala, Giusy Tiseo, Marco Falcone, Mirjam Lista, Monica Poscente, Oreste De Vivo, Paola Petrocelli, Alessandra Guarnaccia, Silvia Baroni, Valentina Perticaroli, Simone Furini, Simona Dei, Elisa Benetti, Nicola Picchiotti, Maurizio Sanarico, Stefano Ceri, Pietro Pinoli, Francesco Raimondi, Filippo Biscarini, Alessandra Stella, Mattia Bergomi, Kristina Zguro, Katia Capitani, Marco Tanfoni, Chiara Fallerini, Sergio Daga, Margherita Baldassarri, Francesca Fava, Elisa Frullanti, Floriana Valentino, Gabriella Doddato, Annarita Giliberti, Rossella Tita, Sara Amitrano, Mirella Bruttini, Susanna Croci, Ilaria Meloni, Maria Antonietta Mencarelli, Caterina Lo Rizzo, Anna Maria Pinto, Giada Beligni, Andrea Tommasi, Laura Di Sarno, Maria Palmieri, Miriam Lucia Carriero, Diana Alaverdian, Nicola Iuso, Gabriele Inchingolo, Stefano Busani, Raffaele Bruno, Marco Vecchia, Mary Ann Belli, Stefania Mantovani, Serena Ludovisi, Eugenia Quiros-Roldan, Melania Degli Antoni, Isabella Zanella, Matteo Siano, Arianna Emiliozzi, Massimiliano Fabbiani, Barbara Rossetti, Giacomo Zanelli, Laura Bergantini, Miriana D’Alessandro, Paolo Cameli, David Bennet, Federico Anedda, Simona Marcantonio, Sabino Scolletta, Susanna Guerrini, Edoardo Conticini, Bruno Frediani, Chiara Spertilli, Alice Donati, Luca Guidelli, Marta Corridi, Leonardo Croci, Paolo Piacentini, Elena Desanctis, Silvia Cappelli, Agnese Verzuri, Valentina Anemoli, Alessandro Pancrazi, Maria Lorubbio, Esther Merlini, Federica Gaia Miraglia, Sophie Venturelli, Andrea Cossarizza, Alessandra Vergori, Arianna Gabrieli, Agostino Riva, Francesca Andretta, Francesca Gatti, Saverio Giuseppe Parisi, Stefano Baratti, Carmelo Piscopo, Roberta Russo, Immacolata Andolfo, Achille Iolascon, Massimo Carella, Giuseppe Merla, Gabriella Maria Squeo, Pamela Raggi, Carmen Marciano, Rita Perna, Matteo Bassetti, Maurizio Sanguinetti, Alessia Giorli, Lorenzo Salerni, Pierpaolo Parravicini, Elisabetta Menatti, Tullio Trotta, Gabriella Coiro, Fabio Lena, Enrico Martinelli, Sandro Mancarella, Chiara Gabbi, Franco Maggiolo, Diego Ripamonti, Tiziana Bachetti, Claudia Suardi, Gianfranco Parati, Giordano Bottà, Paolo Di Domenico, Ilaria Rancan, Francesco Bianchi, Riccardo Colombo, Genes &amp; Health, David A van Heel, Karen A Hunt, Richard C Trembath, Qin Qin Huang, Hilary C Martin, Dan Mason, Bhavi Trivedi, John Wright, Sarah Finer, Genes &amp; Health Research Team*, Christopher J Griffiths, Genes &amp; Health Research Team, Shaheen Akhtar, Mohammad Anwar, Elena Arciero, Samina Ashraf, Gerome Breen, Raymond Chung, Charles J Curtis, Maharun Chowdhury, Grainne Colligan, Panos Deloukas, Ceri Durham, Chris Griffiths, Matt Hurles, Shapna Hussain, Kamrul Islam, Ahsan Khan, Amara Khan, Cath Lavery, Sang Hyuck Lee, Robin Lerner, Daniel MacArthur, Bev MacLaughlin, Hilary Martin, Shefa Miah, Bill Newman, Nishat Safa, Farah Tahmasebi, Genes for Good, Albert V Smith, Andrew P Boughton, Kevin W Li, Jonathon LeFaive, Aubrey Annis, Genetic determinants of COVID-19 complications in the Brazilian population, Cinthia E Jannes, Jose E Krieger, Alexandre C Pereira, Mariliza Velho, Emanuelle Marques, Isabella Ramos Lima, Mauricio Teruo Tada, Karina Valino, Genetic influences on severity of COVID-19 illness in Korea, Mark McCarthy, Carrie Rosenberger, Jong Eun Lee, Diana Chang, Christian Hammer, Julie Hunkapiller, Anubha Mahajan, Sarah Pendergrass, Lara Sucheston-Campbell, Brian Yaspan, Hyun Soo Lee, Eunsoon Shin, Hye Yoon Jang, Sunmie Kim, Sungmin Kym, Yeon-Sook Kim, Hyeongseok Jeong, Ki Tae Kwon, Shin-Woo Kim, Jin Yong Kim, Young Rock Jang, Hyun ah Kim, Ji Yeon Lee, Jeong Eun Lee, Shinwon Lee, Kang-Won Choe, Yu Min Kang, Sun Ha Jee, Keum Ji Jung, Genomic epidemiology of SARS-CoV-2host genetics in coronavirus disease 2019, Victoria Parikh, Euan Ashley, Matthew Wheeler, Manuel Rivas, Carlos Bustamante, Benjamin Pinksy, Phillip Febbo, Kyle Farh, Gary P Schroth, Francis deSouza, Karen Dalton, Jeff Christle, Christopher Deboever, Sándor Szalma, Simone Rubinacci, Olivier Delaneau, John Gorzynski, Hannah de Jong, Shirley Sutton, Nathan Youlton, Ruchi Joshi, David Jimenez-Morales, Christopher Hughes, David Amar, Alex Ioannidis, Steve Hershman, Anna Kirillova, Kinya Seo, Yong Huang, Massa Shoura, Nathan Hammond, Nathaniel Watson, Archana Raja, ChunHong Huang, Malaya Sahoo, Hannah Wang, Jimmy Zhen, Genotek COVID-19 study, Alexander Rakitko, Valery Ilinsky, Danat Yermakovich, Iaroslav Popov, Alexander Chernitsov, Elena Kovalenko, Anna Krasnenko, Nikolay Plotnikov, Ivan Stetsenko, Anna Kim, Helix &amp; Healthy Nevada Project Exome+ COVID-19 Phenotypes, Elizabeth T Cirulli, Kelly M Schiabor Barrett, Alexandre Bolze, Simon White, Nicole L Washington, James T Lu, Stephen Riffle, Francisco Tanudjaja, Xueqing Wang, Jimmy M Ramirez, Nicole Leonetti, Efren Sandoval, Iva Neveux, Shaun Dabe, Joseph J Grzymski, Juan Ignacio Esteban Miñano, Luis A Aguirre, Eduardo López-Collazo, Manuel de la Mata Pazos, Luciano Cerrato, Roberto Lozano-Rodríguez, José Avendaño-Ortiz, Verónica Terrón Arcos, Karla Marina Montalbán-Hernández, Jaime Valentín Quiroga, Alejandro Pascual-Iglesias, Charbel Maroun-Eid, Alejandro Martín-Quirós, Japan Coronavirus Taskforce, Ho Namkoong, Seiya Imoto, Kazuhiko Katayama, Koichi Fukunaga, Yuko Kitagawa, Toshiro Sato, Naoki Hasegawa, Atsushi Kumanogoh, Akinori Kimura, Masumi Ai, Katsushi Tokunaga, Takanori Kanai, Satoru Miyano, Seishi Ogawa, Ryuya Edahiro, Kyuto Sonehara, Yuya Shirai, Makoto Ishii, Hiroki Kabata, Katsunori Masaki, Hirofumi Kamata, Shinnosuke Ikemura, Shotaro Chubachi, Satoshi Okamori, Hideki Terai, Hiromu Tanaka, Atsuho Morita, Ho Lee, Takanori Asakura, Junichi Sasaki, Hiroshi Morisaki, Yoshifumi Uwamino, Kosaku Nanki, Yohei Mikami, Kazunori Tomono, Kazuto Kato, Fumihiko Matsuda, Meiko Takahashi, Nobuyuki Hizawa, Yoshito Takeda, Haruhiko Hirata, Takayuki Shiroyama, Satoru Miyawaki, Ken Suzuki, Yuichi Maeda, Takuro Nii, Yoshimi Noda, Takayuki Niitsu, Yuichi Adachi, Takatoshi Enomoto, Saori Amiya, Reina Hara, Kunihiko Takahashi, Tatsuhiko Anzai, Takanori Hasegawa, Satoshi Ito, Ryuji Koike, Akifumi Endo, Yuji Uchimura, Yasunari Miyazaki, Takayuki Honda, Tomoya Tateishi, Shuji Tohda, Naoya Ichimura, Kazunari Sonobe, Chihiro Sassa, Jun Nakajima, Yasuhito Nannya, Yosuke Omae, Kazuhisa Takahashi, Norihiro Harada, Makoto Hiki, Haruhi Takagi, Ai Nakamura, Etsuko Tagaya, Masatoshi Kawana, Ken Arimura, Takashi Ishiguro, Noboru Takayanagi, Taisuke Isono, Yotaro Takaku, Kenji Takano, Ryusuke Anan, Yukiko Nakajima, Yasushi Nakano, Kazumi Nishio, Soichiro Ueda, Reina Hayashi, Hiroki Tateno, Isano Hase, Shuichi Yoshida, Shoji Suzuki, Keiko Mitamura, Fumitake Saito, Tetsuya Ueda, Masanori Azuma, Tadao Nagasaki, Yoshinori Yasui, Yoshinori Hasegawa, Yoshikazu Mutoh, Takashi Yoshiyama, Tomohisa Shoko, Mitsuaki Kojima, Tomohiro Adachi, Motonao Ishikawa, Kenichiro Takahashi, Kazuyoshi Watanabe, Tadashi Manabe, Fumimaro Ito, Takahiro Fukui, Yohei Funatsu, Hidefumi Koh, Yoshihiro Hirai, Hidetoshi Kawashima, Atsuya Narita, Kazuki Niwa, Yoshiyuki Sekikawa, Fukuki Saito, Kazuhisa Yoshiya, Tomoyuki Yoshihara, Yusuke Suzuki, Sohei Nakayama, Keita Masuzawa, Koichi Nishi, Masaru Nishitsuji, Maiko Tani, Takashi Inoue, Toshiyuki Hirano, Keigo Kobayashi, Naoki Miyazawa, Yasuhiro Kimura, Reiko Sado, Takashi Ogura, Hideya Kitamura, Kota Murohashi, Ichiro Nakachi, Rie Baba, Daisuke Arai, Satoshi Fuke, Hiroshi Saito, Naota Kuwahara, Akiko Fujiwara, Takenori Okada, Tomoya Baba, Junya Noda, Shuko Mashimo, Kazuma Yagi, Tetsuya Shiomi, Mizuha Hashiguchi, Toshio Odani, Takao Mochimaru, Yoshitaka Oyamada, Nobuaki Mori, Namiki Izumi, Kaoru Nagata, Reiko Taki, Koji Murakami, Mitsuhiro Yamada, Hisatoshi Sugiura, Kentaro Hayashi, Tetsuo Shimizu, Yasuhiro Gon, Shigeki Fujitani, Tomoya Tsuchida, Toru Yoshida, Takashi Kagaya, Toshiyuki Kita, Satoru Sakagami, Yoshifumi Kimizuka, Akihiko Kawana, Yoshihiko Nakamura, Hiroyasu Ishikura, Tohru Takata, Takahide Kikuchi, Daisuke Taniyama, Morio Nakamura, Nobuhiro Kodama, Yasunari Kaneyama, Shunsuke Maeda, Yoji Nagasaki, Masaki Okamoto, Sayoko Ishihara, Akihiro Ito, Yusuke Chihara, Mayumi Takeuchi, Keisuke Onoi, Naozumi Hashimoto, Keiko Wakahara, Akira Ando, Makoto Masuda, Aya Wakabayashi, Hiroki Watanabe, Hisako Sageshima, Taka-Aki Nakada, Ryuzo Abe, Tadanaga Shimada, Kodai Kawamura, Kazuya Ichikado, Kenta Nishiyama, Masaki Yamasaki, Satoru Hashimoto, Yu Kusaka, Takehiko Ohba, Susumu Isogai, Minoru Takada, Hidenori Kanda, Yuko Komase, Fumiaki Sano, Koichiro Asano, Tsuyoshi Oguma, Masahiro Harada, Takeshi Takahashi, Takayuki Shibusawa, Shinji Abe, Yuta Kono, Yuki Togashi, Takehiro Izumo, Minoru Inomata, Nobuyasu Awano, Shinichi Ogawa, Tomouki Ogata, Shoichiro Ishihara, Arihiko Kanehiro, Shinji Ozaki, Yasuko Fuchimoto, Yuichiro Kitagawa, Shozo Yoshida, Shinji Ogura, Kei Nishiyama, Kousuke Yoshida, Satoru Beppu, Satoru Fukuyama, Yoshihiro Eriguchi, Akiko Yonekawa, Yoshiaki Inoue, Kunihiro Yamagata, Shigeru Chiba, Osamu Narumoto, Hideaki Nagai, Nobuharu Ooshima, Mitsuru Motegi, Hironori Sagara, Akihiko Tanaka, Shin Ohta, Yoko Shibata, Yoshinori Tanino, Yuki Sato, Yuichiro Yamada, Takuya Hashino, Masato Shinoki, Hajime Iwagoe, Tomonori Imamura, Akira Umeda, Hisato Shimada, Mayu Endo, Shinichi Hayashi, Mai Takahashi, Shigefumi Nakano, Masakiyo Yatomi, Toshitaka Maeno, Tomoo Ishii, Mitsuyoshi Utsugi, Akihiro Ono, Kensuke Kanaoka, Shoichi Ihara, Kiyoshi Komuta, Lifelines, Marike Boezen, Annique Claringbould, Esteban Lopera, Robert Warmerdam, Judith M Vonk, Irene van Blokland, Pauline Lanting, Anil P S Ori, Lung eQTL Consortium, Ma’en Obeidat, Ana I Hernández Cordero, Don D Sin, Yohan Bossé, Philippe Joubert, Ke Hao, David Nickle, Wim Timens, Maarten van den Berge, Mass General Brigham-Host Vulnerability to COVID-19, Yen-Chen Anne Feng, Josep Mercader, Scott T Weiss, Elizabeth W Karlson, Jordan W Smoller, Shawn N Murphy, James B Meigs, Ann E Woolley, Robert C Green, Emma F Perez, Michigan Genomics Initiative (MGI), Sebastian Zöllner, Jiongming Wang, Andrew Beck, Mount Sinai Health System COVID-19 Genomics Initiative, Laura G Sloofman, Steven Ascolillo, Robert P Sebra, Brett L Collins, Tess Levy, Stuart C Sealfon, Daniel M Jordan, Ryan C Thompson, Kyle Gettler, Kumardeep Chaudhary, Gillian M Belbin, Michael Preuss, Clive Hoggart, Sam Choi, Slayton J Underwood, Irene Salib, Bari Britvan, Katherine Keller, Lara Tang, Michael Peruggia, Liam L Hiester, Kristi Niblo, Alexandra Aksentijevich, Alexander Labkowsky, Avromie Karp, Menachem Zlatopolsky, Marissa Zyndorf, Alexander W Charney, Noam D Beckmann, Eric E Schadt, Noura S Abul-Husn, Judy H Cho, Yuval Itan, Eimear E Kenny, Ruth J F Loos, Girish N Nadkarni, Ron Do, Paul O’Reilly, Laura M Huckins, MyCode Health Initiative, Manuel A R Ferreira, Goncalo R Abecasis, Joseph B Leader, Michael N Cantor, Anne E Justice, Dave J Carey, Geetha Chittoor, Navya Shilpa Josyula, Jack A Kosmicki, Julie E Horowitz, Aris Baras, Matthew C Gass, Ashish Yadav, Tooraj Mirshahi, Netherlands Twin Register, Jouke Jan Hottenga, Meike Bartels, Eco J C de Geus, Michel G Nivard, Penn Medicine Biobank, Anurag Verma, Marylyn D Ritchie, Daniel Rader, Binglan Li, Shefali S Verma, Anastasia Lucas, Yuki Bradford, Population controls, Federico Zara, Vincenzo Salpietro, Marcello Scala, Michele Iacomino, Paolo Scudieri, Renata Bocciardi, Carlo Minetti, Antonella Riva, Maria Stella Vari, Jean-François Rahier, Elisa Giorgio, Diana Carli, Edouad Louis, Cynthia M Bulik, Mikael Landén, Alfredo Brusco, Giovanni Battista Ferrero, Francesca Madia, Bengt Fundín, Qatar Genome Program, Said I Ismail, Chadi Saad, Yaser Al-Sarraj, Radja Messai Badji, Wadha Al-Muftah, Asma Al Thani, Nahla Afifi, Study of the COVID-19 host genetics in the population of Latvia, Janis Klovins, Vita Rovite, Raimonds Rescenko, Raitis Peculis, Monta Ustinova, The genetic predisposition to severe COVID-19, Hugo Zeberg, Robert Frithiof, Michael Hultström, Miklos Lipcsey, UCLA Precision Health COVID-19 Host Genomics Biobank, Ruth Johnson, UCLA Health ATLAS &amp; Data Mart Working Group*, UCLA Health ATLAS &amp; Data Mart Working Group, Daniel H Geschwind, Nelson Freimer, Manish J Butte, Yi Ding, Alec Chiu, Timothy S Chang, Paul Boutros, 000 Genomes Project (Genomics England), Loukas Moutsianas, Mark J Caulfield, Richard H Scott, Susan Walker, Alex Stuckey, Christopher A Odhams, Daniel Rhodes, Tom Fowler, Augusto Rendon, Georgia Chan, Prabhu Arumugam, UK Biobank, Konrad J Karczewski, Daniel J Wilson, Chris A Spencer, Derrick W Crook, David H Wyllie, Anne Marie O’Connell, Elizabeth G Atkinson, Kristin Tsuo, Nikolas Baya, Patrick Turley, Rahul Gupta, Raymond K Walters, Duncan S Palmer, Gopal Sarma, Nathan Cheng, Wenhan Lu, Claire Churchhouse, Jacqueline I Goldstein, Daniel King, Cotton Seed, Mark J Daly, Benjamin M Neale, Sam Bryant, F Kyle Satterstrom, Gavin Band, Sarah G Earle, Shang-Kuan Lin, Nicolas Arning, Jacob Armstrong, Justine K Rudkin, Shawneequa Callier, Caroline Cusick, UK Blood Donors Cohort, Nicole Soranzo, Jing Hua Zhao, John Danesh, Emanuele Di Angelantonio, Adam S Butterworth, VA Million Veteran Program (MVP), Yan V Sun, Jennifer E Huffman, Kelly Cho, Christopher J O’Donnell, Phil Tsao, J Michael Gaziano, Gina Peloso, Yuk-Lam Ho, Val Gardena, Michael Mian, Federica Scaggiante, CHOP_CAG, Xiao Chang, Joseph R Glessner, Hakon Hakonarson, GenOMICC/ISARIC4C, Peter J McGuigan, Luke Stephen Prockter Moore, Marcela Paola Vizcaychipi, Kathryn Hall, Andy Campbell, Ailstair Nichol, Geraldine Ward, Valerie Joan Page, Malcolm G Semple, Kayode Adeniji, Daniel Agranoff, Ken Agwuh, Dhiraj Ail, Erin L Aldera, Ana Alegria, Brian Angus, Abdul Ashish, Dougal Atkinson, Shahedal Bari, Gavin Barlow, Stella Barnass, Nicholas Barrett, Christopher Bassford, Sneha Basude, David Baxter, Michael Beadsworth, Jolanta Bernatoniene, John Berridge, Nicola Best, Pieter Bothma, David Chadwick, Robin Brittain-Long, Naomi Bulteel, Tom Burden, Andrew Burtenshaw, Vikki Caruth, Duncan Chambler, Nigel Chee, Jenny Child, Srikanth Chukkambotla, Tom Clark, Paul Collini, Catherine Cosgrove, Jason Cupitt, Maria-Teresa Cutino-Moguel, Paul Dark, Chris Dawson, Samir Dervisevic, Phil Donnison, Sam Douthwaite, Andrew Drummond, Ingrid DuRand, Ahilanadan Dushianthan, Tristan Dyer, Cariad Evans, Chi Eziefula, Christopher Fegan, Adam Finn, Duncan Fullerton, Sanjeev Garg, Atul Garg, Effrossyni Gkrania-Klotsas, Jo Godden, Arthur Goldsmith, Clive Graham, Elaine Hardy, Stuart Hartshorn, Daniel Harvey, Peter Havalda, Daniel B Hawcutt, Maria Hobrok, Luke Hodgson, Anil Hormis, Michael Jacobs, Susan Jain, Paul Jennings, Agilan Kaliappan, Vidya Kasipandian, Stephen Kegg, Michael Kelsey, Jason Kendall, Caroline Kerrison, Ian Kerslake, Oliver Koch, Gouri Koduri, George Koshy, Shondipon Laha, Steven Laird, Susan Larkin, Tamas Leiner, Patrick Lillie, James Limb, Vanessa Linnett, Jeff Little, Mark Lyttle, Michael MacMahon, Emily MacNaughton, Ravish Mankregod, Huw Masson, Elijah Matovu, Katherine McCullough, Ruth McEwen, Manjula Meda, Gary H Mills, Jane Minton, Karl Ward, Mariyam Mirfenderesky, Kavya Mohandas, Quen Mok, James Moon, Elinoor Moore, Patrick Morgan, Craig Morris, Katherine Mortimore, Samuel Moses, Mbiye Mpenge, Rohinton Mulla, Michael Murphy, Megan Nagel, Thapas Nagarajan, Mark Nelson, Matthew K O’Shea, Igor Otahal, Marlies Ostermann, Mark Pais, Selva Panchatsharam, Danai Papakonstantinou, Hassan Paraiso, Brij Patel, Natalie Pattison, Justin Pepperell, Mark Peters, Mandeep Phull, Stefania Pintus, Jagtur Singh Pooni, Frank Post, David Price, Rachel Prout, Nikolas Rae, Henrik Reschreiter, Tim Reynolds, Neil Richardson, Mark Roberts, Devender Roberts, Alistair Rose, Guy Rousseau, Brendan Ryan, Taranprit Saluja, Aarti Shah, Prad Shanmuga, Anil Sharma, Anna Shawcross, Jeremy Sizer, Manu Shankar-Hari, Richard Smith, Catherine Snelson, Nick Spittle, Nikki Staines, Tom Stambach, Richard Stewart, Pradeep Subudhi, Tamas Szakmany, Kate Tatham, Jo Thomas, Chris Thompson, Robert Thompson, Ascanio Tridente, Darell Tupper-Carey, Mary Twagira, Andrew Ustianowski, Nick Vallotton, Lisa Vincent-Smith, Shico Visuvanathan, Alan Vuylsteke, Sam Waddy, Rachel Wake, Andrew Walden, Ingeborg Welters, Tony Whitehouse, Paul Whittaker, Ashley Whittington, Padmasayee Papineni, Meme Wijesinghe, Martin Williams, Lawrence Wilson, Sarah Cole, Stephen Winchester, Martin Wiselka, Adam Wolverson, Daniel G Wooton, Andrew Workman, Bryan Yates, Peter Young, Rupert Beale, Andrew D Bretherick, Sara Clohisey, Max Head Fourman, James Furniss, Elvina Gountouna, Graeme Grimes, Chris Haley, David Harrison, Caroline Hayward, Sean Keating, Lucija Klaric, Paul Klenerman, Andy Law, Alison M Meynert, Jonathan Millar, Erola Pairo-Castineira, Nicholas Parkinson, Chris P Ponting, David J Porteous, Konrad Rawlik, Anne Richmond, Kathy Rowan, Clark D Russell, Xia Shen, Barbara Shih, Albert Tenesa, Veronique Vitart, Bo Wang, James F Wilson, Yang Wu, Jian Yang, Zhijian Yang, Marie Zechner, Ranran Zhai, Chenqing Zheng, Lisa Norman, Riinu Pius, Thomas M Drake, Cameron J Fairfield, Stephen R Knight, Kenneth A Mclean, Derek Murphy, Catherine A Shaw, Jo Dalton, Michelle Girvan, Egle Saviciute, Stephanie Roberts, Janet Harrison, Laura Marsh, Marie Connor, Sophie Halpin, Clare Jackson, Carrol Gamble, Gary Leeming, Andrew Law, Murray Wham, Ross Hendry, James Scott-Brown, Colin Begg, Charles Hinds, Antonia Ying Wai Ho, Peter W Horby, Julian Knight, Lowell Ling, David Maslove, Danny McAuley, Hugh Montgomery, Alistair Nichol, Peter J M Openshaw, Charlotte Summers, Timothy Walsh, Lisa Armstrong, Hayley Bates, Emma Dooks, Fiona Farquhar, Brigid Hairsine, C McParland, Sophie Packham, Zoe Alldis, Raine Astin-Chamberlain, Fatima Bibi, Jack Biddle, Sarah Blow, Matthew Bolton, Catherine Borra, Ruth Bowles, Maudrian Burton, Yasmin Choudhury, David Collier, Amber Cox, Amy Easthope, Patrizia Ebano, Stavros Fotiadis, Jana Gurasashvili, Rosslyn Halls, Pippa Hartridge, Delordson Kallon, Jamila Kassam, Ivone Lancoma-Malcolm, Maninderpal Matharu, Peter May, Oliver Mitchelmore, Tabitha Newman, Mital Patel, Jane Pheby, Irene Pinzuti, Zoe Prime, Oleksandra Prysyazhna, Julian Shiel, Melanie Taylor, Carey Tierney, Suzanne Wood, Anne Zak, Olivier Zongo, Miranda Forsey, Anne Nicholson, Joanne Riches, Mark Vertue, Christopher Wasson, Stephanie Finn, Jackie Green, Erin Collins, Bernadette King, Lina Grauslyte, Musarat Hussain, Tatiana Pogreban, Lace Rosaroso, Erika Salciute, George Franke, Joanna Wong, Aparna George, Louise Akeroyd, Shereen Bano, Matt Bromley, Lucy Gurr, Tom Lawton, James Morgan, Kirsten Sellick, Deborah Warren, Brian Wilkinson, Janet McGowan, Camilla Ledgard, Amelia Stacey, Kate Pye, Ruth Bellwood, Michael Bentley, Ronda Loosley, Heather McGuinness, Helen Tench, Rebecca Wolf-Roberts, Sian Gibson, Amanda Lyle, Fiona McNeela, Jayachandran Radhakrishnan, Alistair Hughes, Asifa Ali, Megan Brady, Sam Dale, Annalisa Dance, Lisa Gledhill, Jill Greig, Kathryn Hanson, Kelly Holdroyd, Marie Home, Diane Kelly, Ross Kitson, Lear Matapure, Deborah Melia, Samantha Mellor, Tonicha Nortcliffe, Jez Pinnell, Matthew Robinson, Lisa Shaw, Ryan Shaw, Lesley Thomis, Alison Wilson, Tracy Wood, Lee-Ann Bayo, Ekta Merwaha, Tahira Ishaq, Sarah Hanley, David Antcliffe, Dorota Banach, Stephen Brett, Phoebe Coghlan, Ziortza Fernandez, Anthony Gordon, Roceld Rojo, Sonia Sousa Arias, Maie Templeton, Rajeev Jha, Vinodh Krishnamurthy, Lai Lim, Rehana Bi, Barney Scholefield, Lydia Ashton, Alison Williams, Claire Cheyne, Anne Saunderson, Angela Allan, Felicity Anderson, Callum Kaye, Jade Liew, Jasmine Medhora, Teresa Scott, Erin Trumper, Adriana Botello, Petra Polgarova, Katerina Stroud, Eoghan Meaney, Megan Jones, Anthony Ng, Shruti Agrawal, Nazima Pathan, Deborah White, Esther Daubney, Kay Elston, Robert Parker, Amie Reddy, Ian Turner-Bone, Laura Wilding, Peter Harding, Reni Jacob, Cathy Jones, Craig Denmade, Maria Croft, Ian White, Li Lim, Denise Griffin, Nycola Muchenje, Mcdonald Mupudzi, Richard Partridge, Jo-Anna Conyngham, Rachel Thomas, Mary Wright, Maria Alvarez Corral, Victoria Bastion, Daphene Clarke, Beena David, Harriet Kent, Rachel Lorusso, Gamu Lubimbi, Sophie Murdoch, Melchizedek Penacerrada, Alastair Thomas, Jennifer Valentine, Ana Vochin, Retno Wulandari, Brice Djeugam, Joy Dawson, Sweyn Garrioch, Melanie Tolson, Jonathan Aldridge, Laura Gomes de Almeida Martins, Jaime Carungcong, Sarah Beavis, Katie Dale, Rachel Gascoyne, Joanne Hawes, Kelly Pritchard, Lesley Stevenson, Amanda Whileman, Anne Cowley, Judith Highgate, Rikki Crawley, Abigail Crew, Mishell Cunningham, Allison Daniels, Laura Harrison, Susan Hope, Ken Inweregbu, Sian Jones, Nicola Lancaster, Jamie Matthews, Alice Nicholson, Gemma Wray, Leonie Benham, Zena Bradshaw, Joanna Brown, Melanie Caswell, Sarah Melling, Stephen Preston, Nicola Slawson, Emma Stoddard, Scott Warden, Edward Combes, Teishel Joefield, Sonja Monnery, Valerie Beech, Sallyanne Trotman, Bridget Hopkins, James Scriven, Laura Thrasyvoulou, Heather Willis, Susan Anderson, Janine Birch, Emma Collins, Kate Hammerton, Ryan O’Leary, Caroline Abernathy, Louise Foster, Andrew Gratrix, Vicky Martinson, Priyai Parkinson, Elizabeth Stones, Llucia Carbral-Ortega, Ritoo Kapoor, David Loader, Karen Castle, Craig Brandwood, Lara Smith, Richard Clark, Katie Birchall, Laurel Kolakaluri, Deborah Baines, Anila Sukumaran, Isheunesu Mapfunde, Megan Meredith, Lucy Morris, Lucy Ryan, Amy Clark, Julia Sampson, Cecilia Peters, Martin Dent, Margaret Langley, Saima Ashraf, Shuying Wei, Angela Andrew, Manish Chablani, Amy Kirkby, Kimberley Netherton, Michelle Bates, Jo Dasgin, Jaspret Gill, Annette Nilsson, Elena Apetri, Cathrine Basikolo, Bethan Blackledge, Laura Catlow, Bethan Charles, Reece Doonan, Jade Harris, Alice Harvey, Daniel Horner, Karen Knowles, Stephanie Lee, Diane Lomas, Chloe Lyons, Tracy Marsden, Danielle McLaughlan, Liam McMorrow, Jessica Pendlebury, Jane Perez, Maria Poulaka, Nicola Proudfoot, Melanie Slaughter, Kathryn Slevin, Vicky Thomas, Danielle Walker, Angiy Michael, Matthew Collis, Martyn Clark, Martina Coulding, Edward Jude, Jacqueline McCormick, Oliver Mercer, Darsh Potla, Hafiz Rehman, Heather Savill, Victoria Turner, Miriam Davey, David Golden, Rebecca Seaman, Jodie Hunt, Joy Dearden, Emma Dobson, Michelle Mulcahy, Sheila Munt, Grainne O’Connor, Jennifer Philbin, Chloe Rishton, Redmond Tully, Sarah Winnard, Lenka Cagova, Adama Fofano, Lucie Garner, Helen Holcombe, Sue Mepham, Alice Michael Mitchell, Lucy Mwaura, K Praman, Alain Vuylsteke, Julie Zamikula, Georgia Bercades, David Brealey, Ingrid Hass, Niall MacCallum, Gladys Martir, Eamon Raith, Anna Reyes, Deborah Smyth, Abigail Taylor, Rachel Anne Hughes, Helen Thomas, Alun Rees, Michaela Duskova, Janet Phipps, Suzanne Brooks, Michelle Edwards, Peter Alexander, Schvearn Allen, Joanne Bradley-Potts, Craig Brantwood, Jasmine Egan, Timothy Felton, Grace Padden, Luke Ward, Stuart Moss, Susannah Glasgow, Kate Beesley, Sarah Board, Agnieszka Kubisz-Pudelko, Alison Lewis, Jess Perry, Lucy Pippard, Di Wood, Clare Buckley, Alison Brown, Jane Gregory, Susan O’Connell, Tim Smith, Zakaula Belagodu, Bridget Fuller, Anca Gherman, Olumide Olufuwa, Remi Paramsothy, Carmel Stuart, Naomi Oakley, Charlotte Kamundi, David Tyl, Katy Collins, Pedro Silva, June Taylor, Laura King, Charlotte Coates, Maria Crowley, Phillipa Wakefield, Jane Beadle, Laura Johnson, Janet Sargeant, Madeleine Anderson, Catherine Jardine, Dewi Williams, Victoria Parris, Sheena Quaid, Ekaterina Watson, Julie Melville, Jay Naisbitt, Rosane Joseph, Maria Lazo, Olivia Walton, Alan Neal, Michaela Hill, Thogulava Kannan, Laura Wild, Elizabeth Allan, Kate Darlington, Ffyon Davies, Jack Easton, Sumit Kumar, Richard Lean, Daniel Menzies, Richard Pugh, Xinyi Qiu, Llinos Davies, Hannah Williams, Jeremy Scanlon, Gwyneth Davies, Callum Mackay, Joannne Lewis, Stephanie Rees, Samantha Coetzee, Alistair Gales, Meena Raj, Craig Sell, Helen Langton, Malcolm Watters, Catherine Novis, Gill Arbane, Aneta Bociek, Sara Campos, Neus Grau, Tim Owen Jones, Rosario Lim, Martina Marotti, Christopher Whitton, Anthony Barron, Ciara Collins, Sundeep Kaul, Heather Passmore, Claire Prendergast, Anna Reed, Paula Rogers, Rajvinder Shokkar, Meriel Woodruff, Hayley Middleton, Oliver Polgar, Claire Nolan, Vicky Thwaites, Kanta Mahay, Chunda Sri-Chandana, Joslan Scherewode, Lorraine Stephenson, Sarah Marsh, Hollie Bancroft, Mary Bellamy, Margaret Carmody, Jacqueline Daglish, Faye Moore, Joanne Rhodes, Mirriam Sangombe, Salma Kadiri, Amanda Ayers, Wendy Harrison, Julie North, Anna Cavazza, Maeve Cockrell, Eleanor Corcoran, Maria Depante, Clare Finney, Ellen Jerome, Mark McPhail, Monalisa Nayak, Harriet Noble, Kevin O’Reilly, Evita Pappa, Rohit Saha, Sian Saha, John Smith, Abigail Knighton, Mandy Gill, Paul Paul, Valli Ratnam, Sarah Shelton, Inez Wynter, David Baptista, Rebecca Crowe, Rita Fernandes, Rosaleen Herdman-Grant, Anna Joseph, Adam Loveridge, India McKenley, Eriko Morino, Andres Naranjo, Richard Simms, Kathryn Sollesta, Andrew Swain, Harish Venkatesh, Jacyntha Khera, Jonathan Fox, Russell Barber, Claire Hewitt, Annette Hilldrith, Karen Jackson-Lawrence, Sarah Shepardson, Maryanne Wills, Susan Butler, Silvia Tavares, Amy Cunningham, Julia Hindale, Sarwat Arif, Linsha George, Sophie Twiss, David Wright, Maureen Holland, Natalie Keenan, Marc Lyons, Helen Wassall, Chris Marsh, Mervin Mahenthran, Emma Carter, Thomas Kong, Oluronke Adanini, Nikhil Bhatia, Maines Msiska, Louise Mew, Esther Mwaura, Felicity Williams, Lynn Wren, Sara-Beth Sutherland, Ceri Battle, Elaine Brinkworth, Rachel Harford, Carl Murphy, Luke Newey, Tabitha Rees, Marie Williams, Sophie Arnold, John Hardy, Henry Houlden, Eleanor Moncur, Ambreen Tariq, Arianna Tucci, Karen Convery, Deirdre Fottrell-Gould, Lisa Hudig, Jocelyn Keshet-Price, Georgina Randell, Katie Stammers, Marwa Abdelrazik, Dhanalakshmi Bakthavatsalam, Munzir Elhassan, Arunkumar Ganesan, Anne Haldeos, Jeronimo Moreno-Cuesta, Dharam Purohit, Rachel Vincent, Kugan Xavier, Kumar Rohit, Frater Alasdair, Malik Saleem, Carter David, Samuel Jenkins, Zoe Lamond, Alanna Wall, Jessica Reynolds, Helen Campbell, Maria Thompsom, Steve Dodds, Stacey Duffy, Deborah Butcher, Susie O’Sullivan, Nicola Butterworth-Cowin, Bethan Deacon, Meg Hibbert, Carla Pothecary, Dariusz Tetla, Christopher Woodford, Latha Durga, Gareth Kennard-Holden, Laura Ortiz-Ruiz de Gordoa, Emily Peasgood, Claire Phillips, Denise Skinner, Jane Gaylard, Dee Mullan, Julie Newman, Ellie Davies, Lisa Roche, Sonia Sathe, Lutece Brimfield, Zoe Daly, David Pogson, Steve Rose, Amy Collins, Waqas Khaliq, Estefania Treus Gude, Louise Allen, Eva Beranova, Nikki Crisp, Joanne Deery, Tracy Hazelton, Alicia Knight, Carly Price, Sorrell Tilbey, Salah Turki, Sharon Turney, Julian Giles, Simon Booth, Gillian Bell, Katy English, Amro Katary, Louise Wilcox, Rachael Campbell, Noreen Clarke, Jonathan Whiteside, Mairi Mascarenhas, Avril Donaldson, Joanna Matheson, Fiona Barrett, Marianne O’Hara, Laura O’Keefe, Clare Bradley, Dawn Collier, Rachel Walker, Victoria Maynard, Tahera Patel, Matthew Smith, Aayesha Kazi, Janice Hartley, Joseph Dykes, Muhammad Hijazi, Sarah Keith, Meherunnisa Khan, Janet Ryan-Smith, Philippa Springle, Jacqueline Thomas, Nick Truman, Samuel Saad, Dabheoc Coleman, Christopher Fine, Roseanna Matt, Bethan Gay, Jack Dalziel, Syamlan Ali, Drew Goodchild, Rhiannan Harling, Ravi Bhatterjee, Wendy Goddard, Chloe Davison, Stephen Duberly, Jeanette Hargreaves, Rachel Bolton, Mark Verlander, Alexandra Williams, Helen Blackman, Ben Creagh-Brown, Sinead Donlon, Natalia Michalak-Glinska, Sheila Mtuwa, Veronika Pristopan, Armorel Salberg, Eleanor Smith, Sarah Stone, Charles Piercy, Jerik Verula, Dorota Burda, Rugia Montaser, Lesley Harden, Irving Mayangao, Cheryl Marriott, Paul Bradley, Celia Harris, Joshua Cooper, Cheryl Finch, Sarah Liderth, Alison Quinn, Natalia Waddington, Katy Fidler, Emma Tagliavini, Kevin Donnelly, Lynn Abel, Michael Brett, Brian Digby, Lisa Gemmell, James Hornsby, Patrick MacGoey, Pauline O’Neil, Richard Price, Natalie Rodden, Kevin Rooney, Radha Sundaram, Nicola Thomson, Rebecca Flanagan, Gareth Hughes, Scott Latham, Emma McKenna, Jennifer Anderson, Robert Hull, Kat Rhead, Debbie Branney, Jordan Frankham, Sally Pitts, Nigel White, Daniele Cristiano, Natalie Dormand, Zohreh Farzad, Mahitha Gummadi, Kamal Liyanage, Brijesh V Patel, Sara Salmi, Geraldine Sloane, Mathew Varghese, Anelise C Zborowski, Sarah Bean, Karen Burt, Michael Spivey, Christine Eastgate-Jackson, Helder Filipe, Daniel Martin, Amitaa Maharajh, Sara Mingo Garcia, Mark De Neef, Ceri Lynch, Gwenllian Sera Howe, Jayaprakash Singh, Keri Turner, Hannah Ellis, Natalie Stroud, Shiney Cherian, Sean Cutler, Anne Emma Heron, Anna Roynon-Reed, Gemma Williams, Owen Richards, Yusuf Cheema, Norfaizan Ahmad, Joann Barker, Kris Bauchmuller, Sarah Bird, Kay Cawthron, Kate Harrington, Yvonne Jackson, Faith Kibutu, Becky Lenagh, Shamiso Masuko, Ajay Raithatha, Matthew Wiles, Jayne Willson, Helen Newell, Alison Lye, Lorenza Nwafor, Claire Jarman, Sarah Rowland-Jones, David Foote, Joby Cole, Roger Thompson, James Watson, Lisa Hesseldon, Irene Macharia, Luke Chetam, Jacqui Smith, Amber Ford, Samantha Anderson, Kathryn Birchall, Kay Housley, Sara Walker, Leanne Milner, Helena Hanratty, Helen Trower, Patrick Phillips, Simon Oxspring, Ben Donne, Emily Bevan, Jane Martin, Dawn Trodd, Geoff Watson, Caroline Wrey Brown, Lara Bunni, Claire Jennings, Monica Latif, Rebecca Marshall, Gayathri Subramanian, Nageswar Bandla, Minnie Gellamucho, Michelle Davies, Christopher Thompson, Fiona Trim, Beena Eapen, Cecilia Ahmed, Balvinder Baines, Sarah Clamp, Julie Colley, Risna Haq, Anne Hayes, Jonathan Hulme, Samia Hussain, Sibet Joseph, Rita Kumar, Zahira Maqsood, Manjit Purewal, Ben Chandler, Kerry Elliott, Janine Mallinson, Alison Turnbull, Kathy Dent, Elizabeth Horsley, Muhmmad Nauman Akhtar, Sandra Pearson, Dorota Potoczna, Sue Spencer, Hayley Blakemore, Borislava Borislavova, Beverley Faulkner, Emma Gendall, Elizabeth Goff, Kati Hayes, Matt Thomas, Ruth Worner, Kerry Smith, Deanna Stephens, Carlos Castro Delgado, Deborah Dawson, Lijun Ding, Georgia Durrant, Obiageri Ezeobu, Sarah Farnell-Ward, Abiola Harrison, Rebecca Kanu, Susannah Leaver, Elena Maccacari, Soumendu Manna, Romina Pepermans Saluzzio, Joana Queiroz, Tinashe Samakomva, Christine Sicat, Joana Texeira, Edna Fernandes Da Gloria, Ana Lisboa, John Rawlins, Jisha Mathew, Ashley Kinch, William James Hurt, Nirav Shah, Victoria Clark, Maria Thanasi, Nikki Yun, Kamal Patel, Vikki Crickmore, Gabor Debreceni, Joy Wilkins, Liz Nicol, Iona Burn, Geraldine Hambrook, Katarina Manso, Ruth Penn, Pradeep Shanmugasundaram, Julie Tebbutt, Danielle Thornton, Anthony Rostron, Alistair Roy, Lindsey Woods, Sarah Cornell, Fiona Wakinshaw, Kimberley Rogerson, Jordan Jarmain, Peter Anderson, Katie Archer, Karen Austin, Caroline Davis, Alison Durie, Olivia Kelsall, Jessica Thrush, Charlie Vigurs, Hannah-Louise Wood, Helen Tranter, Alison Harrison, Nicholas Cowley, Michael McAlindon, Stephen Digby, Emma Low, Aled Morgan, Naiara Cother, Tobias Rankin, Sarah Clayton, Alex McCurdy, Suzanne Allibone, Roman Mary-Genetu, Amit Patel, Ainhi Mac, Anthony Murphy, Parisa Mahjoob, Roonak Nazari, Lucy Worsley, Andrew Fagan, Inthakab Ali Mohamed Ali, Karen Beaumont, Mark Blunt, Zoe Coton, Hollie Curgenven, Mohamed Elsaadany, Kay Fernandes, Sameena Mohamed Ally, Harini Rangarajan, Varun Sarathy, Sivarupan Selvanayagam, Dave Vedage, Matthew White, Jaime Fernandez-Roman, David O Hamilton, Emily Johnson, Brian Johnston, Maria Lopez Martinez, Suleman Mulla, David Shaw, Alicia A C Waite, Victoria Waugh, Ingeborg D Welters, Karen Williams, Thomas Bemand, Ethel Black, Arnold Dela Rosa, Ryan Howle, Shaman Jhanji, Ravishankar Rao Baikady, Kate Colette Tatham, Benjamin Thomas, Matthew Halkes, Pauline Mercer, Lorraine Thornton, Joe West, Tracy Baird, Jim Ruddy, Rosie Reece-Anthony, Mark Birt, Amanda Cowton, Andrea Kay, Melanie Kent, Kathryn Potts, Ami Wilkinson, Suzanne Naylor, Ellen Brown, Michele Clark, Sarah Purvis, Jade Cole, Rhys Davies, Donna Duffin, Helen Hill, Ben Player, Emma Thomas, Angharad Williams, Claire Marie Beith, Karen Black, Suzanne Clements, Alan Morrison, Dominic Strachan, Margaret Taylor, Michelle Clarkson, Stuart D’Sylva, Kathryn Norman, Tina Coventry, Susan Fowler, Amanda McGregor, Ailbhe Brady, Rebekah Chan, Shane McIvor, Helena Prady, Helen Whittle, Bijoy Mathew, Melanie Clapham, Rosemary Harper, Una Poultney, Polly Rice, Rachel Mutch, Yolanda Baird, Aaron Butler, Indra Chadbourn, Linda Folkes, Heather Fox, Amy Gardner, Raquel Gomez, Gillian Hobden, Kirsten King, Michael Margarson, Tim Martindale, Emma Meadows, Dana Raynard, Yvette Thirlwall, David Helm, Jordi Margalef, Sandra Greer, Karen Shuker, Sara Smuts, Joseph Duffield, Oliver Smith, Lewis Mallon, Watkins Claire, Isobel Birkinshaw, Joseph Carter, Kate Howard, Joanne Ingham, Rosie Joy, Harriet Pearson, Samantha Roche, Zoe Scott, Ellen Knights, Alicia Price, Alice Thomas, Chris Thorpe, Azmerelda Abraheem, Peter Bamford, Kathryn Cawley, Charlie Dunmore, Maria Faulkner, Rumanah Girach, Helen Jeffrey, Rhianna Jones, Emily London, Imrun Nagra, Farah Nasir, Hannah Sainsbury, Clare Smedley, Reena Khade, Ashok Sundar, George Tsinaslanidis, Teresa Behan, Caroline Burnett, Jonathan Hatton, Elaine Heeney, Atideb Mitra, Maria Newton, Rachel Pollard, Rachael Stead, Jenny Birch, Laura Bough, Josie Goodsell, Rebecca Tutton, Patricia Williams, Sarah Williams, Barbara Winter-Goodwin, Fiona Auld, Joanne Donnachie, Ian Edmond, Lynn Prentice, Nikole Runciman, Dario Salutous, Lesley Symon, Anne Todd, Patricia Turner, Abigail Short, Laura Sweeney, Euan Murdoch, Dhaneesha Senaratne, Karen Burns, Andrew Higham, Taya Anderson, Dan Hawcutt, Laura O’Malley, Laura Rad, Naomi Rogers, Paula Saunderson, Kathryn Sian Allison, Deborah Afolabi, Jennifer Whitbread, Dawn Jones, Rachael Dore, Liana Lankester, Nikitas Nikitas, Colin Wells, Bethan Stowe, Kayleigh Spencer, Susanne Cathcart, Katharine Duffy, Alex Puxty, Kathryn Puxty, Lynne Turner, Jane Ireland, Gary Semple, Peter Barry, Paula Hilltout, Jayne Evitts, Amanda Tyler, Joanne Waldron, Val Irvine, Benjamin Shelley, Olugbenga Akinkugbe, Alasdair Bamford, Emily Beech, Holly Belfield, Michael Bell, Charlene Davies, Gareth A L Jones, Tara McHugh, Hamza Meghari, Lauran O’Neill, Mark J Peters, Samiran Ray, Ana Luisa Tomas, Claire Gorman, Abhinav Gupta, Elizabeth Timlick, Rebecca Brady, Stephen Bonner, Keith Hugill, Jessica Jones, Steven Liggett, Archana Bashyal, Neil Davidson, Paula Hutton, Stuart McKechnie, Jean Wilson, Neil Flint, Patel Rekha, Dawn Hales, Carina Cruz, Shameer Gopal, Nichola Harris, Victoria Lake, Stella Metherell, Elizabeth Radford, Ian Clement, Bijal Patel, A Gulati, Carole Hays, K Webster, Anne Hudson, Andrea Webster, Elaine Stephenson, Louise McCormack, Victoria Slater, Rachel Nixon, Helen Hanson, Maggie Fearby, Sinead Kelly, Victoria Bridgett, Philip Robinson, Christine Almaden-Boyle, Pauline Austin, Louise Cabrelli, Stephen Cole, Matt Casey, Susan Chapman, Clare Whyte, Adam Brayne, Emma Fisher, Jane Hunt, Peter Jackson, Duncan Kaye, Nicholas Love, Juliet Parkin, Victoria Tuckey, Lynne van Koutrik, Sasha Carter, Benedict Andrew, Louise Findlay, Katie Adams, Michelle Bruce, Karen Connolly, Tracy Duncan, Helen T -Michael, Gabriella Lindergard, Samuel Hey, Claire Fox, Jordan Alfonso, Laura Jayne Durrans, Jacinta Guerin, Martin Hruska, Ayaa Eltayeb, Thomas Lamb, Tracey Hodgkiss, Lisa Cooper, Joanne Rothwell, Catherine Dennis, Alastair McGregor, Sinduya Srikaran, Anisha Sukha, Kim Davies, Linda O’Brien, Zohra Omar, Emma Perkins, Tracy Lewis, Isobel Sutherland, Hollie Brooke, Sarah Buckley, Jose Cebrian Suarez, Ruth Charlesworth, Karen Hansson, John Norris, Alice Poole, Alastair Rose, Rajdeep Sandhu, Brendan Sloan, Elizabeth Smithson, Muthu Thirumaran, Veronica Wagstaff, Alexandra Metcalfe, Julie Camsooksai, Charlotte Humphrey, Sarah Jenkins, Beverley Wadams, Yasmin DeAth, Colene Adams, Anita Agasou, Tracie Arden, Amy Bowes, Pauline Boyle, Mandy Beekes, Heather Button, Nigel Capps, Mandy Carnahan, Anne Carter, Danielle Childs, Denise Donaldson, Kelly Hard, Fran Hurford, Yasmin Hussain, Ayesha Javaid, James Jones, Sanal Jose, Michael Leigh, Terry Martin, Helen Millward, Nichola Motherwell, Rachel Rikunenko, Jo Stickley, Julie Summers, Louise Ting, Helen Tivenan, Louise Tonks, Rebecca Wilcox, Maria Bokhari, Rachael Lucas, Wendy McCormick, Jenny Ritzema, Amanda Sanderson, Helen Wild, Nicola Baxter, Steven Henderson, Sophie Kennedy-Hay, Christopher McParland, Laura Rooney, Malcolm Sim, Gordan McCreath, Mark Brunton, Jess Caterson, Holly Coles, Matthew Frise, Sabi Gurung Rai, Nicola Jacques, Liza Keating, Emma Tilney, Shauna Bartley, Parminder Bhuie, Charlotte Downes, Kathleen Holding, Katie Riches, Mary Hilton, Mel Hayman, Deepak Subramanian, Priya Daniel, Letizia Zitter, Sarah Benyon, Suzie Marriott, Linda Park, Samantha Keenan, Elizabeth Gordon, Helen Quinn, Kizzy Baines, Gillian Andrew, Lucy Barclay, Marie Callaghan, Sarah Clark, Dave Hope, Lucy Marshall, Corrienne McCulloch, Kate Briton, Jo Singleton, Sophie Birch, Kerry Simpson, Jayne Craig, Carrie Demetriou, Charlotte Eckbad, Sarah Hierons, Lucy Howie, Sarah Mitchard, Lidia Ramos, Alfredo Serrano-Ruiz, Katie White, Fiona Kelly, Vishal Amin, Elena Anastasescu, Vikram Anumakonda, Komala Karthik, Rizwana Kausar, Karen Reid, Jacqueline Smith, Janet Imeson-Wood, Arianna Bellini, Jade Bryant, Anton Mayer, Amy Pickard, Nicholas Roe, Jason Sowter, Alex Howlett, Kristine Criste, Rebecca Cusack, Kim Golder, Hannah Golding, Oliver Jones, Samantha Leggett, Michelle Male, Martyna Marani, Kirsty Prager, Toran Williams, Belinda Roberts, Karen Salmon, Prisca Gondo, B Hadebe, Abdul Kayani, Bridgett Masunda, Ashar Ahmed, Anna Morris, Srinivas Jakkula, Kate Long, Simon Whiteley, Elizabeth Wilby, Bethan Ogg, Sam Moultrie, M Odam, Jeremy Bewley, Zoe Garland, Lisa Grimmer, Bethany Gumbrill, Rebekah Johnson, Katie Sweet, Denise Webster, Georgia Efford, Sara Bennett, Emma Goodwin, Matthew Jackson, Alissa Kent, Clare Tibke, Wiesia Woodyatt, Ahmed Zaki, Amelia Daniel, Joanne Finn, Rajnish Saha, Pamela Bremmer, J Allan, T Geary, Gordon Houston, A Meikle, P O’Brien, Dina Bell, Rosalind Boyle, Katie Douglas, Lynn Glass, Emma Lee, Liz Lennon, Austin Rattray, Rob Charnock, Denise McFarland, Denise Cosgrove, Ben Attwood, Penny Parsons, Siobhain Carmody, Metod Oblak, Monica Popescu, Mini Thankachen, Rosie Baruah, Sheila Morris, Susie Ferguson, Amy Shepherd, Abdelhakim Altabaibeh, Ana Alvaro, Kayleigh Gilbert, Louise Ma, Loreta Mostoles, Chetan Parmar, Kathryn Simpson, Champa Jetha, Lauren Booker, Anezka Pratley, Tracey Cosier, Gemma Millen, Natasha Schumacher, Heather Weston, James Rand, Beatrice Alex, Benjamin Bach, Wendy S Barclay, Debby Bogaert, Meera Chand, Graham S Cooke, Annemarie B Docherty, Jake Dunning, Ana da Silva Filipe, Tom Fletcher, Christoper A Green, Ewen M Harrison, Julian A Hiscox, Samreen Ijaz, Saye Khoo, Wei Shen Lim, Alexander J Mentzer, Laura Merson, Mahdad Noursadeghi, Shona C Moore, Massimo Palmarini, William A Paxton, Georgios Pollakis, Nicholas Price, Andrew Rambaut, David L Robertson, Vanessa Sancho-Shimizu, Janet T Scott, Thushan de Silva, Louise Sigfrid, Tom Solomon, Shiranee Sriskandan, David Stuart, Richard S Tedder, Emma C Thomson, A A Roger Thompson, Ryan S Thwaites, Lance C W Turtle, Rishi K Gupta, Carlo Palmieri, Olivia V Swann, Maria Zambon, Marc-Emmanuel Dumas, Julian L Griffin, Zoltan Takats, Kanta Chechi, Petros Andrikopoulos, Anthonia Osagie, Michael Olanipekun, Sonia Liggi, Matthew R Lewis, Gonçalo dos Santos Correia, Caroline J Sands, Panteleimon Takis, Lynn Maslen, William Greenhalf, Victoria Shaw, Sarah E McDonald, Seán Keating, Katie A Ahmed, Jane A Armstrong, Milton Ashworth, Innocent G Asiimwe, Siddharth Bakshi, Samantha L Barlow, Laura Booth, Benjamin Brennan, Katie Bullock, Benjamin W A Catterall, Jordan J Clark, Emily A Clarke, Louise Cooper, Helen Cox, Christopher Davis, Oslem Dincarslan, Chris Dunn, Philip Dyer, Angela Elliott, Anthony Evans, Lorna Finch, Lewis W S Fisher, Terry Foster, Isabel Garcia-Dorival, Philip Gunning, Catherine Hartley, Rebecca L Jensen, Christopher B Jones, Trevor R Jones, Shadia Khandaker, Katharine King, Robyn T Kiy, Chrysa Koukorava, Annette Lake, Suzannah Lant, Diane Latawiec, Lara Lavelle-Langham, Daniella Lefteri, Lauren Lett, Lucia A Livoti, Maria Mancini, Sarah McDonald, Laurence McEvoy, John McLauchlan, Soeren Metelmann, Nahida S Miah, Joanna Middleton, Joyce Mitchell, Ellen G Murphy, Rebekah Penrice-Randal, Jack Pilgrim, Tessa Prince, Will Reynolds, P Matthew Ridley, Debby Sales, Victoria E Shaw, Rebecca K Shears, Benjamin Small, Krishanthi S Subramaniam, Agnieska Szemiel, Aislynn Taggart, Jolanta Tanianis-Hughes, Jordan Thomas, Erwan Trochu, Libby van Tonder, Eve Wilcock, J Eunice Zhang, Lisa Flaherty, Nicole Maziere, Emily Cass, Alejandra Doce Carracedo, Nicola Carlucci, Anthony Holmes, Hannah Massey, Lee Murphy, Nicola Wrobel, Sarah McCafferty, Kirstie Morrice, Alan MacLean, Ruth Armstrong, Ceilia Boz, Adam Brown, Audrey Coutts, Louise Cullum, Nicky Day, Lorna Donnelly, Esther Duncan, Angie Fawkes, Paul Finernan, Tammy Gilchrist, Ailsa Golightly, Katarzyna Hafezi, Dawn Law, Rachel Law, Sarah Law, Louise Macgillivray, Alan Maclean, Hanning Mal, Ellie Mcmaster, Jen Meikle, Wilna Oosthuyzen, Trevor Paterson, Andrew Stenhouse, Maaike Swets, Helen Szoor-McElhinney, Filip Taneski, Tony Wackett, Mairi Ward, Jane Weaver, Judy Coyle, Bernadette Gallagher, Rebecca Lidstone-Scott, Debbie Hamilton, Katherine Schon, Anita Furlong, Heather Biggs, Fiona Griffiths, Eleanor Andrews, Kathy Brickell, Michelle Smyth, Lorna Murphy, Gail Carson, Hayley Hardwick, Chloe Donohue, COVID-19 HGI corresponding authors' : '3609 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3609 more authors</span> </div> <div class="periodical"> <em>Nature</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41586-021-03767-x" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-021-03767-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The genetic make-up of an individual contributes to the susceptibility and response to viral infection. Although environmental, clinical and social factors have a role in the chance of exposure to SARS-CoV-2 and the severity of COVID-191,2, host genetics may also be important. Identifying host-specific genetic factors may reveal biological mechanisms of therapeutic relevance and clarify causal relationships of modifiable environmental risk factors for SARS-CoV-2 infection and outcomes. We formed a global network of researchers to investigate the role of human genetics in SARS-CoV-2 infection and COVID-19 severity. Here we describe the results of three genome-wide association meta-analyses that consist of up to 49,562 patients with COVID-19 from 46 studies across 19 countries. We report 13 genome-wide significant loci that are associated with SARS-CoV-2 infection or severe manifestations of COVID-19. Several of these loci correspond to previously documented associations to lung or autoimmune and inflammatory diseases3–7. They also represent potentially actionable mechanisms in response to infection. Mendelian randomization analyses support a causal role for smoking and body-mass index for severe COVID-19 although not for type II diabetes. The identification of novel host genetic factors associated with COVID-19 was made possible by the community of human genetics researchers coming together to prioritize the sharing of data, results, resources and analytical frameworks. This working model of international collaboration underscores what is possible for future genetic discoveries in emerging pandemics, or indeed for any complex human disease. A global network of researchers was formed to investigate the role of human genetics in SARS-CoV-2 infection and COVID-19 severity; this paper reports 13 genome-wide significant loci and potentially actionable mechanisms in response to infection.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2021.png"></div> <div id="10.1002/ana.25918" class="col-sm-8"> <div class="title">Causal Associations Between Modifiable Risk Factors and the Alzheimer’s Phenome</div> <div class="author"> Shea J Andrews, Brian Fulton‐Howard, Paul O’Reilly, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Edoardo Marcora, Alison M Goate, Lindsay A Farrer, Jonathan L Haines, Richard Mayeux, Adam C Naj, Margaret A Pericak‐Vance, Gerard D Schellenberg, Li‐San Wang' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Annals of Neurology</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/ana.25918" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/ana.25918" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The purpose of this study was to infer causal relationships between 22 previously reported risk factors for Alzheimer’s disease (AD) and the “AD phenome”: AD, AD age of onset (AAOS), hippocampal volume, cortical surface area and thickness, cerebrospinal fluid (CSF) levels of amyloid‐β (Aβ42), tau, and ptau181, and the neuropathological burden of neuritic plaques, neurofibrillary tangles (NFTs), and vascular brain injury (VBI). Polygenic risk scores (PRS) for the 22 risk factors were computed in 26,431 AD cases/controls and the association with AD was evaluated using logistic regression. Two‐sample Mendelian randomization (MR) was used to infer the causal effect of risk factors on the AD phenome. PRS for increased education and diastolic blood pressure were associated with reduced risk for AD. MR indicated that only education was causally associated with reduced risk of AD, delayed AAOS, and increased cortical surface area and thickness. Total‐ and LDL‐cholesterol levels were causally associated with increased neuritic plaque burden, although the effects were driven by single nucleotide polymorphisms (SNPs) within the APOE locus. Diastolic blood pressure and pulse pressure are causally associated with increased risk of VBI. Furthermore, total cholesterol was associated with decreased hippocampal volume; smoking initiation with decreased cortical thickness; type 2 diabetes with an earlier AAOS; and sleep duration with increased cortical thickness. Our comprehensive examination of the genetic evidence for the causal relationships between previously reported risk factors in AD using PRS and MR supports a causal role for education, blood pressure, cholesterol levels, smoking, and diabetes with the AD phenome. ANN NEUROL 2021;89:54–65</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Novikova2021.png"></div> <div id="10.1186/s13024-021-00449-0" class="col-sm-8"> <div class="title">Beyond association: successes and challenges in linking non-coding genetic variation to functional consequences that modulate Alzheimer’s disease risk</div> <div class="author"> Gloriia Novikova, Shea J. Andrews, Alan E. Renton, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Edoardo Marcora' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Molecular Neurodegeneration</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13024-021-00449-0" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13024-021-00449-0" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Alzheimer’s disease (AD) is the most common type of dementia, affecting millions of people worldwide; however, no disease-modifying treatments are currently available. Genome-wide association studies (GWASs) have identified more than 40 loci associated with AD risk. However, most of the disease-associated variants reside in non-coding regions of the genome, making it difficult to elucidate how they affect disease susceptibility. Nonetheless, identification of the regulatory elements, genes, pathways and cell type/tissue(s) impacted by these variants to modulate AD risk is critical to our understanding of disease pathogenesis and ability to develop effective therapeutics. In this review, we provide an overview of the methods and approaches used in the field to identify the functional effects of AD risk variants in the causal path to disease risk modification as well as describe the most recent findings. We first discuss efforts in cell type/tissue prioritization followed by recent progress in candidate causal variant and gene nomination. We discuss statistical methods for fine-mapping as well as approaches that integrate multiple levels of evidence, such as epigenomic and transcriptomic data, to identify causal variants and risk mechanisms of AD-associated loci. Additionally, we discuss experimental approaches and data resources that will be needed to validate and further elucidate the effects of these variants and genes on biological pathways, cellular phenotypes and disease risk. Finally, we discuss future steps that need to be taken to ensure that AD GWAS functional mapping efforts lead to novel findings and bring us closer to finding effective treatments for this devastating disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/McInerney2021.png"></div> <div id="10.1186/s12859-021-04337-8" class="col-sm-8"> <div class="title">A globally diverse reference alignment and panel for imputation of mitochondrial DNA variants</div> <div class="author"> Tim W McInerney, Brian Fulton-Howard, Christopher Patterson, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Devashi Paliwal, Lars S Jermiin, Hardip R Patel, Judy Pa, Russell H Swerdlow, Alison Goate, Simon Easteal, Shea J Andrews, for the Alzheimer’s Disease Neuroimaging Initiative' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>BMC Bioinformatics</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s12859-021-04337-8" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12859-021-04337-8" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Variation in mitochondrial DNA (mtDNA) identified by genotyping microarrays or by sequencing only the hypervariable regions of the genome may be insufficient to reliably assign mitochondrial genomes to phylogenetic lineages or haplogroups. This lack of resolution can limit functional and clinical interpretation of a substantial body of existing mtDNA data. To address this limitation, we developed and evaluated a large, curated reference alignment of complete mtDNA sequences as part of a pipeline for imputing missing mtDNA single nucleotide variants (mtSNVs). We call our reference alignment and pipeline MitoImpute. We aligned the sequences of 36,960 complete human mitochondrial genomes downloaded from GenBank, filtered and controlled for quality. These sequences were reformatted for use in imputation software, IMPUTE2. We assessed the imputation accuracy of MitoImpute by measuring haplogroup and genotype concordance in data from the 1000 Genomes Project and the Alzheimer’s Disease Neuroimaging Initiative (ADNI). The mean improvement of haplogroup assignment in the 1000 Genomes samples was 42.7% (Matthew’s correlation coefficient = 0.64). In the ADNI cohort, we imputed missing single nucleotide variants. These results show that our reference alignment and panel can be used to impute missing mtSNVs in existing data obtained from using microarrays, thereby broadening the scope of functional and clinical investigation of mtDNA. This improvement may be particularly useful in studies where participants have been recruited over time and mtDNA data obtained using different methods, enabling better integration of early data collected using less accurate methods with more recent sequence data.</p> </div> </div> </div> </li> </ol> <h2 class="year">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Burns2020.png"></div> <div id="10.1192/bjo.2020.29" class="col-sm-8"> <div class="title">Role of apolipoprotein E epsilon 4 (APOE*ε4) as an independent risk factor for incident depression over a 12-year period in cognitively intact adults across the lifespan</div> <div class="author"> Richard Andrew Burns, <em>Shea Andrews</em>, Nicolas Cherbuin, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Kaarin Jane Anstey' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>BJPsych Open</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1192/bjo.2020.29" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1192/bjo.2020.29" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The apolipoprotein E ε4 allele (APOE*ε4) is indicated as a risk for Alzheimer’s disease and other age-related diseases. The risk attributable to APOE*ε4 for depression is less clear and may be because of confounding of the relationship between dementia and depression. We examined the risk of APOE* ε4 for incident depression and depressive symptomology over a 12-year period across the adult lifespan. Participants were from the Personality and Total Health Through Life study, aged 20 to 24 (n = 1420), 40 to 44 (n = 1592) or 60–64 (n = 1768) at baseline, and interviewed every 4 years since 1999. Ethnicities other than White, those without genotyping and those with depression at baseline, or who reported strokes and scores on the Mini-Mental State Examination &lt;27 at any observation, were excluded. Over the study period, there was no evidence that APOE*ε4+ was a risk factor for depression, including any depression (odds ratio (OR) = 0.94, 95% CI 0.77–1.16, P = 0.573), major depression (OR = 0.96, 95% CI 0.60–1.53, P = 0.860), minor depression (OR = 0.94, 95% CI 0.67–1.30, P = 0.695) or depressive symptomology (incidence rate ratio (IRR) = 1.02, 95% CI 0.97–1.08, P = 0.451). APOE*ε4 was unrelated to incident depression. Findings were consistent for all age cohorts. Among cognitively intact Australian adults who were free of depression at baseline, there was little evidence that APOE*ε4+ carriers are at increased risk for depression over a 12-year period among those who are cognitively intact.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2020a.png"></div> <div id="10.1016/j.neurobiolaging.2019.09.007" class="col-sm-8"> <div class="title">Mitonuclear interactions influence Alzheimer’s disease risk</div> <div class="author"> Shea J. Andrews, Brian Fulton-Howard, Christopher Patterson, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'G Peggy McFall, Alden Gross, Elias K. Michaelis, Alison Goate, Russell H. Swerdlow, Judy Pa, Alzheimer’s Disease Neuroimaging Initiative' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Neurobiology of Aging</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2019.09.007" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2019.09.007" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>We examined the associations between mitochondrial DNA haplogroups (MT-hgs; mitochondrial haplotype groups defined by a specific combination of single nucleotide polymorphisms labeled as letters running from A to Z) and their interactions with a polygenic risk score composed of nuclear-encoded mitochondrial genes (nMT-PRS) with risk of dementia and age of onset (AOO) of dementia. MT-hg K (Odds ratio [OR]: 2.03 [95% CI: 1.04, 3.97]) and a 1 SD larger nMT-PRS (OR: 2.2 [95% CI: 1.68, 2.86]) were associated with elevated odds of dementia. Significant antagonistic interactions between the nMT-PRS and MT-hg K (OR: 0.45 [95% CI: 0.22, 0.9]) and MT-hg T (OR: 0.22 [95% CI: 0.1, 0.49]) were observed. Individual MT-hgs were not associated with AOO; however, a significant antagonistic interactions was observed between the nMT-PRS and MT-hg T (Hazard ratio: 0.62 [95% CI: 0.42, 0.91]) and a synergistic interaction between the nMT-PRS and MT-hg V (Hazard ratio: 2.28 [95% CI: 1.19, 4.35]). These results suggest that MT-hgs influence dementia risk and that variants in the nuclear and mitochondrial genome interact to influence the AOO of dementia.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Dumitrescu2020.png"></div> <div id="10.1093/brain/awaa209" class="col-sm-8"> <div class="title">Genetic variants and functional pathways associated with resilience to Alzheimer’s disease</div> <div class="author"> Logan Dumitrescu, Emily R Mahoney, Shubhabrata Mukherjee, and <span class="more-authors" title="click to view 47 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '47 more authors' ? 'Michael L Lee, William S Bush, Corinne D Engelman, Qiongshi Lu, David W Fardo, Emily H Trittschuh, Jesse Mez, Catherine Kaczorowski, Hector Hernandez Saucedo, Keith F Widaman, Rachel Buckley, Michael Properzi, Elizabeth Mormino, Hyun-Sik Yang, Tessa Harrison, Trey Hedden, Kwangsik Nho, Shea J Andrews, Doug Tommet, Niran Hadad, R Elizabeth Sanders, Douglas M Ruderfer, Katherine A Gifford, Annah M Moore, Francis Cambronero, Xiaoyuan Zhong, Neha S Raghavan, Badri Vardarajan, The Alzheimer’s Disease Neuroimaging Initiative (ADNI), A4 Study Team, Margaret A Pericak-Vance, Lindsay A Farrer, Li-San Wang, Carlos Cruchaga, Gerard Schellenberg, Nancy J Cox, Jonathan L Haines, C Dirk Keene, Andrew J Saykin, Eric B Larson, Reisa A Sperling, Richard Mayeux, David A Bennett, Julie A Schneider, Paul K Crane, Angela L Jefferson, Timothy J Hohman' : '47 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">47 more authors</span> </div> <div class="periodical"> <em>Brain</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/brain/awaa209" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/brain/awaa209" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Approximately 30% of older adults exhibit the neuropathological features of Alzheimer’s disease without signs of cognitive impairment. Yet, little is known about the genetic factors that allow these potentially resilient individuals to remain cognitively unimpaired in the face of substantial neuropathology. We performed a large, genome-wide association study (GWAS) of two previously validated metrics of cognitive resilience quantified using a latent variable modelling approach and representing better-than-predicted cognitive performance for a given level of neuropathology. Data were harmonized across 5108 participants from a clinical trial of Alzheimer’s disease and three longitudinal cohort studies of cognitive ageing. All analyses were run across all participants and repeated restricting the sample to individuals with unimpaired cognition to identify variants at the earliest stages of disease. As expected, all resilience metrics were genetically correlated with cognitive performance and education attainment traits (P-values &lt; 2.5 × 10−20), and we observed novel correlations with neuropsychiatric conditions (P-values &lt; 7.9 × 10−4). Notably, neither resilience metric was genetically correlated with clinical Alzheimer’s disease (P-values &gt; 0.42) nor associated with APOE (P-values &gt; 0.13). In single variant analyses, we observed a genome-wide significant locus among participants with unimpaired cognition on chromosome 18 upstream of ATP8B1 (index single nucleotide polymorphism rs2571244, minor allele frequency = 0.08, P = 2.3 × 10−8). The top variant at this locus (rs2571244) was significantly associated with methylation in prefrontal cortex tissue at multiple CpG sites, including one just upstream of ATPB81 (cg19596477; P = 2 × 10−13). Overall, this comprehensive genetic analysis of resilience implicates a putative role of vascular risk, metabolism, and mental health in protection from the cognitive consequences of neuropathology, while also providing evidence for a novel resilience gene along the bile acid metabolism pathway. Furthermore, the genetic architecture of resilience appears to be distinct from that of clinical Alzheimer’s disease, suggesting that a shift in focus to molecular contributors to resilience may identify novel pathways for therapeutic targets.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Swerdlow2020.png"></div> <div id="10.1002/alz.12119" class="col-sm-8"> <div class="title">Exploratory analysis of mtDNA haplogroups in two Alzheimer’s longitudinal cohorts</div> <div class="author"> Russell H. Swerdlow, Dongwei Hui, Prabhakar Chalise, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Palash Sharma, Xinkun Wang, Shea J. Andrews, Judy Pa, Jonathan D. Mahnken, Jill Morris, Heather M. Wilkins, Jeffrey M. Burns, Mary L. Michaelis, Elias K. Michaelis, For the Alzheimer’s Disease Neuroimaging Initiative (ADNI)' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>Alzheimer’s &amp; Dementia</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/alz.12119" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/alz.12119" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Inherited mitochondrial DNA (mtDNA) variants may influence Alzheimer’s disease (AD) risk. We sequenced mtDNA from 146 AD and 265 cognitively normal (CN) subjects from the University of Kansas AD Center (KUADC) and assigned haplogroups. We further considered 244 AD and 242 CN AD Neuroimaging Initiative (ADNI) subjects with equivalent data. Without applying multiple comparisons corrections, KUADC haplogroup J AD and CN frequencies were 16.4% versus 7.6% (P = .007), and haplogroup K AD and CN frequencies were 4.8% versus 10.2% (P = .063). ADNI haplogroup J AD and CN frequencies were 10.7% versus 7.0% (P = .20), and haplogroup K frequencies were 4.9% versus 8.7% (P = .11). For the combined 390 AD and 507 CN cases haplogroup J frequencies were 12.8% versus 7.3% (P = .006), odds ratio (OR) = 1.87, and haplogroup K frequencies were 4.9% versus 9.5% (P = .010), OR = 0.49. Associations remained significant after adjusting for apolipoprotein E, age, and sex. This exploratory analysis suggests inherited mtDNA variants influence AD risk.</p> </div> </div> </div> </li> </ol> <h2 class="year">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2019a.png"></div> <div id="10.1007/s40142-019-0156-2" class="col-sm-8"> <div class="title">Protective Variants in Alzheimer’s Disease</div> <div class="author"> Shea J. Andrews, Brian Fulton-Howard, and Alison Goate</div> <div class="periodical"> <em>Current Genetic Medicine Reports</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1007/s40142-019-0156-2" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1007/s40142-019-0156-2" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Over the last decade, over 40 loci have been associated with risk of Alzheimer’s disease (AD). However, most studies have either focused on identifying risk loci or performing unbiased screens without a focus on protective variation in AD. Here, we provide a review of known protective variants in AD and their putative mechanisms of action. Additionally, we recommend strategies for finding new protective variants. Recent Genome-Wide Association Studies have identified both common and rare protective variants associated with AD. These include variants in or near APP, APOE, PLCG2, MS4A, MAPT-KANSL1, RAB10, ABCA1, CCL11, SORL1, NOCT, SCL24A4-RIN3, CASS4, EPHA1, SPPL2A, and NFIC. There are very few protective variants with functional evidence and a derived allele with a frequency below 20%. Additional fine mapping and multi-omic studies are needed to further validate and characterize known variants as well as specialized genome-wide scans to identify novel variants.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Pa2019.png"></div> <div id="10.1016/j.tins.2019.08.003" class="col-sm-8"> <div class="title">Mitochondria and Alzheimer’s: Is PTCD1 the Smoking Gun?</div> <div class="author"> Judy Pa, Shea J. Andrews, and Russell H. Swerdlow</div> <div class="periodical"> <em>Trends in Neurosciences</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.tins.2019.08.003" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.tins.2019.08.003" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Multiple features distinguish the mitochondria of Alzheimer’s disease (AD) patients from those of unaffected individuals. The causes and consequences of these differences are informed by the recent finding by Fleck et al. (J. Neurosci., 2019) that pentatricopeptide repeat-containing protein 1 (PTCD1) gene variants associate with AD. This suggests a proximal or initiating role for mitochondria in AD.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2019b.png"></div> <div id="10.1016/j.neurobiolaging.2019.09.003" class="col-sm-8"> <div class="title">Mendelian randomization indicates that TNF is not causally associated with Alzheimer’s disease</div> <div class="author"> Shea J. Andrews, and Alison Goate</div> <div class="periodical"> <em>Neurobiology of Aging</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2019.09.003" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2019.09.003" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Epidemiological research has suggested that inhibition of tumor necrosis factor (TNF)-α in patients with rheumatoid arthritis (RA) reduces the overall risk of Alzheimer’s disease (AD). TNF-α antagonists have been suggested as a potential treatment for AD. We used a two-sample Mendelian randomization design to examine the causal relationship between blood TNF expression, serum TNF-α levels, and RA on AD risk. Our results do not support a causal relationship between TNF expression, serum TNF-α levels, and RA on AD risk.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Peters2019.png"></div> <div id="10.1212/wnl.0000000000008732" class="col-sm-8"> <div class="title">Investigation of antihypertensive class, dementia, and cognitive decline: A meta-analysis</div> <div class="author"> Ruth Peters, Sevil Yasar, Craig S Anderson, and <span class="more-authors" title="click to view 56 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '56 more authors' ? 'Shea Andrews, Riitta Antikainen, Hisatomi Arima, Nigel Beckett, Joanne C Beer, Anne Suzanne Bertens, Andrew Booth, Martin van Boxtel, Carol Brayne, Henry Brodaty, Michelle C Carlson, John Chalmers, Maria Corrada, Steven DeKosky, Carol Derby, Roger A Dixon, Françoise Forette, Mary Ganguli, Willem A van Gool, Antonio Guaita, Ann M Hever, David B Hogan, Carol Jagger, Mindy Katz, Claudia Kawas, Patrick G Kehoe, Sirkka Keinanen-Kiukaanniemi, Rose Ann Kenny, Sebastian Köhler, Setor K Kunutsor, Jari Laukkanen, Colleen Maxwell, G Peggy McFall, Tessa van Middelaar, Eric P Moll van Charante, Tze-Pin Ng, Jean Peters, Iris Rawtaer, Edo Richard, Kenneth Rockwood, Lina Rydén, Perminder S Sachdev, Ingmar Skoog, Johan Skoog, Jan A Staessen, Blossom C M Stephan, Sylvain Sebert, Lutgarde Thijs, Stella Trompet, Phillip J Tully, Christophe Tzourio, Roberta Vaccaro, Eeva Vaaramo, Erin Walsh, Jane Warwick, Kaarin J Anstey' : '56 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">56 more authors</span> </div> <div class="periodical"> <em>Neurology</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1212/wnl.0000000000008732" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1212/wnl.0000000000008732" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Objective High blood pressure is one of the main modifiable risk factors for dementia. However, there is conflicting evidence regarding the best antihypertensive class for optimizing cognition. Our objective was to determine whether any particular antihypertensive class was associated with a reduced risk of cognitive decline or dementia using comprehensive meta-analysis including reanalysis of original participant data. Methods To identify suitable studies, MEDLINE, Embase, and PsycINFO and preexisting study consortia were searched from inception to December 2017. Authors of prospective longitudinal human studies or trials of antihypertensives were contacted for data sharing and collaboration. Outcome measures were incident dementia or incident cognitive decline (classified using the reliable change index method). Data were separated into mid and late-life (&gt;65 years) and each antihypertensive class was compared to no treatment and to treatment with other antihypertensives. Meta-analysis was used to synthesize data. Results Over 50,000 participants from 27 studies were included. Among those aged &gt;65 years, with the exception of diuretics, we found no relationship by class with incident cognitive decline or dementia. Diuretic use was suggestive of benefit in some analyses but results were not consistent across follow-up time, comparator group, and outcome. Limited data precluded meaningful analyses in those ≤65 years of age. Conclusion Our findings, drawn from the current evidence base, support clinical freedom in the selection of antihypertensive regimens to achieve blood pressure goals. Clinical trials registration The review was registered with the international prospective register of systematic reviews (PROSPERO), registration number CRD42016045454.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2019c.png"></div> <div id="10.3233/jad-190342" class="col-sm-8"> <div class="title">Association of Alzheimer’s Disease Genetic Risk Loci with Cognitive Performance and Decline: A Systematic Review</div> <div class="author"> Shea J Andrews, G Peggy McFall, Andrew Booth, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Roger A Dixon, Kaarin J Anstey' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Journal of Alzheimer’s Disease</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.3233/jad-190342" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.3233/jad-190342" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The association of Apolipoprotein E (APOE) with late-onset Alzheimer’s disease (LOAD) and cognitive endophenotypes of aging has been widely investigated. There is increasing interest in evaluating the association of other LOAD risk loci with cognitive performance and decline. The results of these studies have been inconsistent and inconclusive. We conducted a systematic review of studies investigating the association of non-APOE LOAD risk loci with cognitive performance in older adults. Studies published from January 2009 to April 2018 were identified through a PubMed database search using keywords and by scanning reference lists. Studies were included if they were either cross-sectional or longitudinal in design, included at least one genome-wide significant LOAD risk loci or a genetic risk score, and had one objective measure of cognition. Quality assessment of the studies was conducted using the quality of genetic studies (Q-Genie) tool. Of 2,466 studies reviewed, 49 met inclusion criteria. Fifteen percent of the associations between non-APOE LOAD risk loci and cognition were significant. However, these associations were not replicated across studies, and the majority were rendered non-significant when adjusting for multiple testing. One-third of the studies included genetic risk scores, and these were typically significant only when APOE was included. The findings of this systematic review do not support a consistent association between individual non-APOE LOAD risk and cognitive performance or decline. However, evidence suggests that aggregate LOAD genetic risk exerts deleterious effects on decline in episodic memory and global cognition.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2019c.png"></div> <div id="10.1016/j.jalz.2019.09.086" class="col-sm-8"> <div class="title">Association between alcohol consumption and Alzheimer’s disease: A Mendelian randomization study</div> <div class="author"> Shea J. Andrews, Alison Goate, and Kaarin J. Anstey</div> <div class="periodical"> <em>Alzheimer’s &amp; Dementia</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.jalz.2019.09.086" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.jalz.2019.09.086" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Introduction Observational studies have suggested that light-to-moderate alcohol consumption decreases the risk of Alzheimer’s disease, but it is unclear if this association is causal. Methods Two-sample Mendelian randomization (MR) analysis was used to examine whether alcohol consumption, alcohol dependence, or Alcohol Use Disorder Identification Test (AUDIT) scores were causally associated with the risk of Late-Onset Alzheimer’s disease (LOAD) or Alzheimer’s disease age of onset survival (AAOS). Additionally, γ-glutamyltransferase levels were included as a positive control. Results There was no evidence of a causal association between alcohol consumption, alcohol dependence, or AUDIT, and LOAD. Alcohol consumption was associated with an earlier AAOS and increased γ-glutamyltransferase blood concentrations. Alcohol dependence was associated with a delayed AAOS. Discussion MR found robust evidence of a causal association between alcohol consumption and an earlier AAOS, but not alcohol intake and LOAD risk. The protective effect of alcohol dependence is potentially due to survivor bias.</p> </div> </div> </div> </li> </ol> <h2 class="year">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Watts2018.png"></div> <div id="10.1159/000486369" class="col-sm-8"> <div class="title">Sex Differences in the Impact of BDNF Genotype on the Longitudinal Relationship between Physical Activity and Cognitive Performance</div> <div class="author"> Amber Watts, Shea J. Andrews, and Kaarin J. Anstey</div> <div class="periodical"> <em>Gerontology</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1159/000486369" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1159/000486369" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Physical activity may preserve cognitive function in older adults, but benefits vary by sex and genetic factors. Objective: We tested the longitudinal association between physical activity and cognitive performance to de termine whether a common genetic polymorphism for brain-derived neurotrophic factor (BDNF Val66Met) moderated this effect. Methods: In a 12-year longitudinal population-based sample of older adults (n = 2,218), we used growth curve modeling to investigate whether the benefits of physical activity on cognitive preservation differed by BDNF genotype and sex across multiple cognitive domains including processing speed, attention, working memory, and episodic verbal memory. Results: The relationship between physical activity and cognitive performance was dependent on BDNF carrier status in males (Δχ2 [Δdf] = 12.94 [4], p = 0.01), but not in females (Δχ2 [Δdf] = 4.38 [4], p = 0.36). Cognition benefited from physical activity in male BDNF met noncarriers, but not met carriers, whereas cognition was not statistically significantly related to physical activity in females regardless of genotype. Conclusion: We observed longitudinal, but not cross-sectional, effects of physical activity on cognitive performance. Our study highlights the importance of longitudinal follow-up and consideration of sex differences in the relationships between physical activity, BDNF genotype, and cognitive decline. The findings contribute to understanding gene-lifestyle interactions in promoting cognitive health.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2018a.png"></div> <div id="10.1002/ajmg.b.32684" class="col-sm-8"> <div class="title">Association of Alzheimer’s genetic loci with mild behavioral impairment</div> <div class="author"> Shea J. Andrews, Zahinoor Ismail, Kaarin J. Anstey, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Moyra Mortby' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>American Journal of Medical Genetics Part B: Neuropsychiatric Genetics</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/ajmg.b.32684" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/ajmg.b.32684" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Mild behavioral impairment (MBI) describes the emergence of later‐life neuropsychiatric symptoms (NPS) as an at‐risk state for incident cognitive decline and dementia, and for some as a potential manifestation of prodromal dementia. How NPS mechanistically link to the development of mild cognitive impairment and Alzheimer’s disease (AD) is not fully understood, with potential mechanisms including shared risk factors related to both NPS and cognitive impairment, or AD pathology promoting NPS. This is the first exploratory study to examine whether AD genetic loci as a genetic risk score (GRS), or individually, are a shared risk factor with MBI. Participants were 1,226 older adults (aged 72–79; 738 males; 763 normal cognition) from the Personality and Total Health Through Life project. MBI was approximated in accordance with Criterion 1 of the ISTAART‐AA diagnostic criteria using a transformation algorithm for the neuropsychiatric inventory. A GRS was constructed from 25 AD risk loci. Binomial logistic regression adjusting for age, gender, and education examined the association between GRS and MBI. A higher GRS and APOE*ε4 were associated with increased likelihood of affective dysregulation. Nominally significant associations were observed between MS4A4A‐rs4938933*C and MS4A6A‐rs610932*G with a reduced likelihood of affective dysregulation; ZCWPW1‐rs1476679*C with a reduced likelihood of social inappropriateness and abnormal perception/thought content; BIN1‐rs744373*G and EPHA1‐rs11767557*C with higher likelihood of abnormal perception/thought content; NME8‐rs2718058*G with a reduced likelihood of decreased motivation. These preliminary findings suggest a common genetic etiology between MBI and traditionally recognized cognitive problems observed in AD and improve our understanding of the pathophysiological features underlying MBI.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2018b.png"></div> <div id="10.1097/wad.0000000000000292" class="col-sm-8"> <div class="title">Alzheimer’s Environmental and Genetic Risk Scores are Differentially Associated With General Cognitive Ability and Dementia Severity</div> <div class="author"> Shea J. Andrews, G. Peggy McFall, Roger A. Dixon, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Nicolas Cherbuin, Ranmalee Eramudugolla, Kaarin J. Anstey' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Alzheimer Disease &amp; Associated Disorders</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1097/wad.0000000000000292" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1097/wad.0000000000000292" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>We investigated the association of the Australian National University Alzheimer’s Disease Risk Index (ANU-ADRI) and an Alzheimer disease (AD) genetic risk score (GRS) with cognitive performance. The ANU-ADRI (composed of 12 risk factors for AD) and GRS (composed of 25 AD risk loci) were computed in 1061 community-dwelling older adults. Participants were assessed on 11 cognitive tests and activities of daily living. Structural equation modeling was used to evaluate the association of the ANU-ADRI and GRS with: (1) general cognitive ability (g), (2) dementia-related variance in cognitive performance (δ), and (3) verbal ability (VA), episodic memory (EM), executive function (EF), and processing speed (PS). A worse ANU-ADRI score was associated with poorer performance in “g” [β (SE)=−0.40 (0.02), P&lt;0.001], δ [−0.40 (0.04), P&lt;0.001], and each cognitive domain [VA=−0.29 (0.04), P&lt;0.001; EM=−0.34 (0.03), P&lt;0.001; EF=−0.38 (0.03), P&lt;0.001; and PS=−0.40 (0.03), P&lt;0.001]. A worse GRS was associated with poorer performance in δ [−0.08 (0.03), P=0.041] and EM [−0.10 (0.03), P=0.035]. The ANU-ADRI was broadly associated with worse cognitive performance, including general ability and dementia severity, validating its further use in early dementia risk assessment.</p> </div> </div> </div> </li> </ol> <h2 class="year">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2017a.png"></div> <div id="10.1186/s13195-017-0240-3" class="col-sm-8"> <div class="title">Validating the role of the Australian National University Alzheimer’s Disease Risk Index (ANU-ADRI) and a genetic risk score in progression to cognitive impairment in a population-based cohort of older adults followed for 12 years</div> <div class="author"> Shea J. Andrews, Ranmalee Eramudugolla, Jorge I. Velez, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Nicolas Cherbuin, Simon Easteal, Kaarin J. Anstey' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Alzheimer’s Research &amp; Therapy</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13195-017-0240-3" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13195-017-0240-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The number of people living with dementia is expected to exceed 130 million by 2050, which will have serious personal, social and economic implications. Employing successful intervention and treatment strategies focused on disease prevention is currently the only available approach that can have an impact on the projected rates of dementia, with risk assessment being a key component of population-based risk reduction for identification of at-risk individuals. We evaluated a risk index comprising lifestyle, medical and demographic factors (the Australian National University Alzheimer’s Disease Risk Index [ANU-ADRI]), as well as a genetic risk score (GRS), for assessment of the risk of progression to mild cognitive impairment (MCI). The ANU-ADRI was computed for the baseline assessment of 2078 participants in the Personality and Total Health (PATH) Through Life project. GRSs were constructed on the basis of 25 single-nucleotide polymorphisms previously associated with Alzheimer’s disease (AD). Participants were assessed for clinically diagnosed MCI and dementia as well as psychometric test-based MCI (MCI-TB) at 12 years of follow-up. Multi-state models were used to estimate the odds of transitioning from cognitively normal (CN) to MCI, dementia and MCI-TB over 12 years according to baseline ANU-ADRI and GRS. A higher ANU-ADRI score was associated with increased risk of progressing from CN to both MCI and MCI-TB (HR 1.07 [95% CI 1.04–1.11]; 1.07 [1.04–1.09]). The GRS was associated with transitions from CN to dementia (HR 4.19 [95% CI 1.72–10.20), but not to MCI or MCI-TB (HR 1.05 [95% CI 0.86–1.29]; 1.03 [0.87–1.21]). Limitations of our study include that the ethnicity of participants in the PATH project is predominately Caucasian, potentially limiting the generalisability of the results of this study to people of other ethnicities. Biomarkers of AD were not available to define MCI attributable to AD. Not all the predictive variables for the ANU-ADRI were available in the PATH project. In the general population, the ANU-ADRI, comprising lifestyle, medical and demographic factors, is associated with the risk of progression from CN to MCI, whereas a GRS comprising the main AD risk genes was not associated with this risk. The ANU-ADRI may be used for population-level risk assessment and screening.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2017b.png"></div> <div id="10.3233/jad-160774" class="col-sm-8"> <div class="title">Late Onset Alzheimer’s Disease Risk Variants in Cognitive Decline: The PATH Through Life Study</div> <div class="author"> Shea J. Andrews, Debjani Das, Kaarin J. Anstey, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Simon Easteal' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Journal of Alzheimer’s Disease</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.3233/jad-160774" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.3233/jad-160774" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Recent genome wide association studies have identified a number of single nucleotide polymorphisms associated with late onset Alzheimer’s disease (LOAD). We examined the associations of 24 LOAD risk loci, individually and collectively as a genetic risk score, with cognitive function. We used data from 1,626 non-demented older Australians of European ancestry who were examined up to four times over 12 years on tests assessing episodic memory, working memory, vocabulary, and information processing speed. Linear mixed models were generated to examine associations between genetic factors and cognitive performance. Twelve SNPs were significantly associated with baseline cognitive performance (ABCA7, MS4A4E, SORL1), linear rate of change (APOE, ABCA7, INPP5D, ZCWPW1, CELF1), or quadratic rate of change (APOE, CLU, EPHA1, HLA-DRB5, INPP5D, FERMT2). In addition, a weighted genetic risk score was associated with linear rate of change in episodic memory and information processing speed. Our results suggest that a minority of AD related SNPs may be associated with non-clinical cognitive decline. Further research is required to verify these results and to examine the effect of preclinical AD in genetic association studies of cognitive decline. The identification of LOAD risk loci associated with non-clinical cognitive performance may help in screening for individuals at greater risk of cognitive decline.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2017c.png"></div> <div id="10.1111/gbb.12368" class="col-sm-8"> <div class="title">Association of AKAP6 and MIR2113 with cognitive performance in a population‐based sample of older adults</div> <div class="author"> S. J. Andrews, D. Das, K. J. Anstey, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'S. Easteal' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Genes, Brain and Behavior</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1111/gbb.12368" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1111/gbb.12368" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Genetic factors make a substantial contribution to inter-individual variability in cognitive function. A recent meta-analysis of genome-wide association studies identified two loci, AKAP6 and MIR2113, that are associated with general cognitive function. Here, we extend this previous research by investigating the association of MIR2113 and AKAP6 with baseline and longitudinal non-linear change across a broad spectrum of cognitive domains in a community-based cohort of older adults without dementia. Two single nucleotide polymorphisms (SNPs), MIR211-rs10457441 and AKAP6-rs17522122 were genotyped in 1570 non-demented older Australians of European ancestry, who were examined up to 4 times over 12 years. Linear mixed effects models were used to examine the association between AKAP6 and MIR2113 with cognitive performance in episodic memory, working memory, vocabulary, perceptual speed and reaction time at baseline and with linear and quadratic rates of change. AKAP6-rs17522122*T was associated with worse baseline performance in episodic memory, working memory, vocabulary and perceptual speed, but it was not associated with cognitive change in any domain. MIR2113-rs10457441*T was associated with accelerated decline in episodic memory. No other associations with baseline cognitive performance or with linear or quadratic rate or cognitive changes were observed for this SNP. These results confirm the previous finding that AKAP6 is associated with performance across multiple cognitive domains at baseline but not with cognitive decline, while MIR2113 primarily affects the rate at which memory declines over time.</p> </div> </div> </div> </li> </ol> <h2 class="year">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/McDermott2016.png"></div> <div id="10.1093/geronb/gbw161" class="col-sm-8"> <div class="title">Memory Resilience to Alzheimer’s Genetic Risk: Sex Effects in Predictor Profiles</div> <div class="author"> Kirstie L McDermott, G Peggy McFall, Shea J Andrews, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Kaarin J Anstey, Roger A Dixon' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>The Journals of Gerontology Series B: Psychological Sciences and Social Sciences</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/geronb/gbw161" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/geronb/gbw161" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Apolipoprotein E (APOE) ɛ4 and Clusterin (CLU) C alleles are risk factors for Alzheimer’s disease (AD) and episodic memory (EM) decline. Memory resilience occurs when genetically at-risk adults perform at high and sustained levels. We investigated whether (a) memory resilience to AD genetic risk is predicted by biological and other risk markers and (b) the prediction profiles vary by sex and AD risk variant.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2016.png"></div> <div id="10.1016/j.neurobiolaging.2016.02.016" class="col-sm-8"> <div class="title">Association of genetic risk factors with cognitive decline: the PATH through life project</div> <div class="author"> Shea J. Andrews, Debjani Das, Nicolas Cherbuin, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Kaarin J. Anstey, Simon Easteal' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Neurobiology of Aging</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2016.02.016" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2016.02.016" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p> We examined the association of 28 single nucleotide polymorphisms (SNPs), previously associated with dementia or cognitive performance, with tests assessing episodic memory, working memory, vocabulary, and perceptual speed in 1689 nondemented older Australians of European ancestry. In addition to testing each variant individually, we assessed the collective association of the 12-risk SNPs for late-onset Alzheimer’s disease using weighted and unweighted genetic risk scores. Significant associations with cognitive performance were observed for APOE ε4 allele, ABCA7-rs3764650, CR1-rs3818361, MS4A4E-rs6109332, BDNF-rs6265, COMT-rs4680, CTNNBL-rs6125962, FRMD4A-rs17314229, FRMD4A-rs17314229, intergenic SNP chrX-rs12007229, PDE7A-rs10808746, SORL1-rs668387, and ZNF224-rs3746319. In addition, the weighted genetic risk score was associated with worse performance on episodic memory. The identification of genetic risk factors, that act individually or collectively, may help in screening for people with elevated risk of cognitive decline and for understanding the biological pathways that underlie cognitive decline.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Velez2016.png"></div> <div id="10.1038/mp.2015.177" class="col-sm-8"> <div class="title">APOE*E2 allele delays age of onset in PSEN1 E280A Alzheimer’s disease</div> <div class="author"> J I Vélez, F Lopera, D Sepulveda-Falla, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'H R Patel, A S Johar, A Chuah, C Tobón, D Rivera, A Villegas, Y Cai, K Peng, R Arkell, F X Castellanos, S J Andrews, M F Silva Lara, P K Creagh, S Easteal, J de Leon, M L Wong, J Licinio, C A Mastronardi, M Arcos-Burgos' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>Molecular Psychiatry</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/mp.2015.177" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/mp.2015.177" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Alzheimer’s disease (AD) age of onset (ADAOO) varies greatly between individuals, with unique causal mutations suggesting the role of modifying genetic and environmental interactions. We analyzed \textbackslashtextasciitilde50 000 common and rare functional genomic variants from 71 individuals of the ‘Paisa’ pedigree, the world’s largest pedigree segregating a severe form of early-onset AD, who were affected carriers of the fully penetrant E280A mutation in the presenilin-1 (PSEN1) gene. Affected carriers with ages at the extremes of the ADAOO distribution (30s–70s age range), and linear mixed-effects models were used to build single-locus regression models outlining the ADAOO. We identified the rs7412 (APOE*E2 allele) as a whole exome-wide ADAOO modifier that delays ADAOO by \textbackslashtextasciitilde12 years (β=11.74, 95% confidence interval (CI): 8.07–15.41, P=6.31 × 10−8, PFDR=2.48 × 10−3). Subsequently, to evaluate comprehensively the APOE (apolipoprotein E) haplotype variants (E1/E2/E3/E4), the markers rs7412 and rs429358 were genotyped in 93 AD affected carriers of the E280A mutation. We found that the APOE*E2 allele, and not APOE*E4, modifies ADAOO in carriers of the E280A mutation (β=8.24, 95% CI: 4.45–12.01, P=3.84 × 10−5). Exploratory linear mixed-effects multilocus analysis suggested that other functional variants harbored in genes involved in cell proliferation, protein degradation, apoptotic and immune dysregulation processes (i.e., GPR20, TRIM22, FCRL5, AOAH, PINLYP, IFI16, RC3H1 and DFNA5) might interact with the APOE*E2 allele. Interestingly, suggestive evidence as an ADAOO modifier was found for one of these variants (GPR20) in a set of patients with sporadic AD from the Paisa genetic isolate. This is the first study demonstrating that the APOE*E2 allele modifies the natural history of AD typified by the age of onset in E280A mutation carriers. To the best of our knowledge, this is the largest analyzed sample of patients with a unique mutation sharing uniform environment. Formal replication of our results in other populations and in other forms of AD will be crucial for prediction, follow-up and presumably developing new therapeutic strategies for patients either at risk or affected by AD.</p> </div> </div> </div> </li> </ol> <h2 class="year">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2015.png"></div> <div id="10.3233/jad-140630" class="col-sm-8"> <div class="title">Interactive Effect of APOE Genotype and Blood Pressure on Cognitive Decline: The PATH Through Life Study</div> <div class="author"> <em>Shea Andrews</em>, Debjani Das, Kaarin J. Anstey, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Simon Easteal' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Journal of Alzheimer’s Disease</em>, 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.3233/jad-140630" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.3233/jad-140630" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The apolipoprotein E (APOE) *ε4 allele and hypertension are two of the most prevalent risk factors for cognitive decline in later life. Here we investigate whether cognitive decline is affected by interaction between these two risk factors. Specifically, we examine whether APOE*ε4 moderates the association between high blood pressure and cognition in later life. Cognitive function was assessed at three time points over a period of 8 years in 1,474 cognitively normal, community-dwelling adults aged 60–64 years at baseline. Blood pressure and APOE genotype were assessed at baseline. Blood pressure was measured categorically as ‘Hypertension’ and continuously as ‘Mean Arterial Pressure’ (MAP). Multilevel models were used to investigate main and interactive effects of APOE genotype and both hypertension and MAP on the rate of change of episodic memory, working memory, verbal ability, perceptual speed, and global cognition. The APOE–hypertension interaction was associated with a small but statistically significant increase in the rate of decline of episodic memory, verbal ability, and global cognition. However, its inclusion in the model did not increase the amount of outcome variation explained beyond that already explained by the effect of time. In contrast, the APOE-MAP interaction had no effect on the rate of decline in any of these domains of cognitive performance. These results provide tentative evidence that APOE genotype moderates the association between high blood pressure and cognitive decline in later life.</p> </div> </div> </div> </li></ol> <h2 class="year">2014</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/Andrews2014.png"></div> <div id="10.1038/nrg3520" class="col-sm-8"> <div class="title">Emerging evidence for functional peptides encoded by short open reading frames</div> <div class="author"> Shea J. Andrews, and Joseph A. Rothnagel</div> <div class="periodical"> <em>Nature Reviews Genetics</em>, 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/nrg3520" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/nrg3520" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Short open reading frames (sORFs) of ∼100 codons in length are common and are distributed throughout the genome, but not all sORFs are biologically relevant.sORFs are found on non-coding RNAs and within the 5′ leader and 3′ trailer regions of mRNAs. They can also overlap with the main protein-coding sequence of mRNAs.The identification of sORFs that are translatable and that are likely to encode short peptides remains a major challenge. Three complementary approaches that are typically used to discover functional sORFs are bioinformatics, transcriptomics and proteomics.Bioinformatic studies have identified a large pool of potentially translatable sORFs on the basis of sequence characteristics such as degree of conservation, coding potential and context of the initiation codon.Global ribosome profiling has provided evidence of ribosome engagement at the start codon of many sORFs in various species, including yeast, insects, plants and mammals.Proteomic studies using mass spectrometry on size-fractionated whole-cell lysates have identified several short peptides encoded by sORFs (sPEPs) in human tissues and cell lines.Functional sPEPs have been identified in insects, plants and mammals, but only a small number of them have been fully characterized. Short open reading frames (sORFs) of ∼100 codons in length are common and are distributed throughout the genome, but not all sORFs are biologically relevant. sORFs are found on non-coding RNAs and within the 5′ leader and 3′ trailer regions of mRNAs. They can also overlap with the main protein-coding sequence of mRNAs. The identification of sORFs that are translatable and that are likely to encode short peptides remains a major challenge. Three complementary approaches that are typically used to discover functional sORFs are bioinformatics, transcriptomics and proteomics. Bioinformatic studies have identified a large pool of potentially translatable sORFs on the basis of sequence characteristics such as degree of conservation, coding potential and context of the initiation codon. Global ribosome profiling has provided evidence of ribosome engagement at the start codon of many sORFs in various species, including yeast, insects, plants and mammals. Proteomic studies using mass spectrometry on size-fractionated whole-cell lysates have identified several short peptides encoded by sORFs (sPEPs) in human tissues and cell lines. Functional sPEPs have been identified in insects, plants and mammals, but only a small number of them have been fully characterized. Recent advances in bioinformatics, proteomics and high-throughput analyses of translation have begun to uncover short open reading frames (sORFs) with coding potential. This Review considers unifying characteristics of these sORFs, methods to identify them and potential functions of the resultant short peptides. Short open reading frames (sORFs) are a common feature of all genomes, but their coding potential has mostly been disregarded, partly because of the difficulty in determining whether these sequences are translated. Recent innovations in computing, proteomics and high-throughput analyses of translation start sites have begun to address this challenge and have identified hundreds of putative coding sORFs. The translation of some of these has been confirmed, although the contribution of their peptide products to cellular functions remains largely unknown. This Review examines this hitherto overlooked component of the proteome and considers potential roles for sORF-encoded peptides.</p> </div> </div> </div> </li></ol> <h2 class="year">2013</h2> <ol class="bibliography"></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Shea J. Andrews. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>